<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8" />
    <title>Long-term support projects | NBIS - National Bioinformatics Infrastructure Sweden</title>

    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="generator" content="GravCMS" />
    <meta name="description" content="NBIS (National Bioinformatics Infrastructure Sweden) is a distributed national research infrastructure supported by the Swedish Research Council (Vetenskapsrådet), Science for Life Laboratory, all major Swedish universities and the Knut and Alice Wallenberg Foundation, providing state-of-the-art bioinformatics to the Swedish life science researchers community. NBIS is also the Swedish contact point to the European infrastructure for biological information ELIXIR." />

    <link rel="icon" type="image/png" href="/user/themes/quark/images/favicon.png" />
    <link rel="canonical" href="http://localhost/support/ltsprojects" />

                        
            
    <link href="/assets/user/themes/quark/css-compiled/spectre.min.css" type="text/css" rel="stylesheet">
<link href="/assets/user/themes/quark/css-compiled/theme.min.css" type="text/css" rel="stylesheet">
<link href="/assets/user/themes/quark/css/custom.css" type="text/css" rel="stylesheet">
<link href="/assets/user/themes/quark/css/line-awesome.min.css" type="text/css" rel="stylesheet">

    <script src="/assets/system/assets/jquery/jquery-3.x.min.js"></script>

</head>
<body id="top" class=" header-fixed header-animated sticky-footer">
    <div id="page-wrapper">
            <section id="header" class="section">
            <section class="container grid-lg">
                <nav class="navbar">
                    <section class="navbar-section logo">
                        <a href="/" class="navbar-brand mr-10">
  <svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" viewBox="0 0 504 140" clip-rule="evenodd"><path d="M235.83 71.56h-7.98c-1.2 0-2.2 1-2.2 2.2V89.1l-.15.13c-4.7 3.96-10.64 6.14-16.72 6.14-14.36 0-26.04-11.68-26.04-26.04s11.68-26.04 26.04-26.04c5.58 0 10.92 1.76 15.44 5.1.87.66 2.1.57 2.86-.2l5.7-5.7c.44-.44.67-1.05.63-1.68-.02-.62-.32-1.2-.82-1.6-6.76-5.35-15.2-8.3-23.8-8.3-21.18 0-38.42 17.23-38.42 38.4 0 21.2 17.24 38.42 38.42 38.42 10.93 0 21.4-4.7 28.7-12.9.35-.4.55-.93.55-1.47v-19.6c0-1.22-.98-2.2-2.2-2.2M502.8 34.44c-.4-.6-1.1-.98-1.84-.98h-8.7c-.87 0-1.66.52-2 1.32l-24.5 56.84-24.9-56.85c-.36-.8-1.15-1.3-2.02-1.3h-8.72c-.74 0-1.44.36-1.84.98-.4.62-.48 1.4-.17 2.1l30.2 68.85c.34.8 1.13 1.32 2 1.32h11c.88 0 1.67-.53 2.02-1.33l29.66-68.87c.3-.68.22-1.47-.2-2.1"/><path d="M388.68 34.77c-.35-.8-1.14-1.32-2-1.32h-11c-.88 0-1.67.53-2.02 1.33L344 103.64c-.3.68-.22 1.47.18 2.08.4.62 1.1 1 1.84 1h8.7c.86 0 1.66-.53 2-1.33l24.5-56.86 24.9 56.86c.36.8 1.15 1.32 2.02 1.32h8.72c.74 0 1.44-.38 1.84-1 .4-.62.47-1.4.17-2.1l-30.2-68.85zM309.2 81.52l.47-.22c8.68-4.2 14.28-13.1 14.28-22.67 0-13.88-11.3-25.18-25.17-25.18H266.9c-1.2 0-2.2 1-2.2 2.2v68.86c0 1.23 1 2.22 2.2 2.22h8c1.2 0 2.2-1 2.2-2.2V45.8h21.68c7.05 0 12.8 5.75 12.8 12.8 0 5.9-4 11-9.73 12.42-1.04.26-2.07.4-3.07.4h-7.98c-.83 0-1.6.46-1.96 1.2-.38.73-.3 1.62.2 2.3l22.6 30.87c.42.58 1.08.92 1.78.92h9.9c.84 0 1.6-.47 1.97-1.2.37-.75.3-1.64-.2-2.3l-15.9-21.7zM107.2 80.97c-7.26-4.8-11.4-8.85-15.02-16.1-2.47 4.97-8.24 12.37-17.96 18.2-4.86 15.1-27.96 44-35.43 39.9-2.22-1.2-2.64-2.8-2.15-4.45.54-4.13 9.08-13.62 9.08-13.62s.18 2 2.92 6.18c-3.6-11.2 5.96-25.03 8.5-29.73 3.98-1.27 4.27-6.4 4.27-6.4.26-7.9-3.28-13.63-6.7-17.05 2.46 3 3.25 7.54 3.37 11.7v.02c0 .47 0 .93 0 1.4-.12 3.43-1.16 8.18-3.38 8.18v.03c-2.28-.1-5.1.4-7.63 1.18l-5.6 1.34s2.98-.13 4.6 1.25c-1.8 2.9-5.78 6.53-10.22 8.58-6.45 3-8.3-2.96-5.03-6.84.8-.94 1.62-1.74 2.38-2.4-.5-.5-.8-1.2-.88-2.06 0 0 0 0 0-.02-.46-1.97-.2-4.54 2.6-8.62.54-.86 1.2-1.75 2-2.65.02-.04.04-.07.07-.1.03-.04.07-.08.1-.12.02-.02.04-.04.06-.06.2-.23.42-.45.64-.67 3.34-3.4 8.6-6.96 16.9-10.15C64.4 43.68 67.94 41 67.94 41c1.07-1.1 2.94-2.45 3.63-2.8-5.05-8.77-6.07-21.15-4.75-24.5-.1.2-.2.38-.3.57.5-1.14.83-1.5 1.34-2.1 1.38-1.64 6.06-2.5 7.74.96.9 1.84 1.06 4.23 1.03 6.02-3.7-.2-7.06 4.04-7.06 4.04s3.07-1.46 6.88-1.5c0 0 1 .9 2.28 2.56-1.7 3.2-4.52 10.02-2.5 17.16.35 1.4.86 2.62 1.5 3.65.02.05.04.1.07.14.05.07.1.13.14.2 3.37 5.06 9.54 5.66 9.54 5.66-2.9-1.45-5.27-3.76-6.8-6.56-.82-1.5-1.3-2.77-1.6-3.77-1.64-6.3.77-10 2.14-12.47 3.17-4.9 8.95-7.9 15.15-7.18 8.72 1 14.97 8.86 13.98 17.57-.6 5.32-3.78 9.72-8.15 12.12 1.05 2.84-.07 6.28-.07 6.28 2.64 3.32 2.76 5.23 2.67 7-3.36-.55-6.62 1.7-6.62 1.7s6.48-1.53 10.24 1.82c2.44 2.64 4.08 5 5.05 6.77 1.4 2.5 7.86 2.68 7.12 7.2-.74 4.5-5.68 4.53-13.4-.57M69.56 0C31.15 0 0 31.15 0 69.57c0 38.42 31.15 69.57 69.57 69.57 38.42 0 69.57-31.15 69.57-69.57C139.14 31.15 108 0 69.57 0M73.8 51.7c.8-.82.8-2.14 0-2.95-.82-.82-2.14-.82-2.95 0-.82.8-.82 2.13 0 2.94.8.8 2.13.8 2.95 0M66.45 53.15c-.82.8-.82 2.13 0 2.95.8.8 2.13.8 2.94 0 .8-.82.8-2.14 0-2.95-.82-.8-2.14-.8-2.95 0"/><path d="M79.23 54.23c-1.27-1.27-3.34-1.27-4.6 0l-2.72 2.7c-1.27 1.3-1.27 3.35 0 4.63l3 2.97c1.26 1.28 3.32 1.28 4.6 0l2.7-2.7c1.28-1.28 1.28-3.35 0-4.62l-2.97-2.97zM95.76 41.44c-2.15-2.57 1.87-7.25 4.4-4.46 4.64 5.15-2.25 7.04-4.4 4.46m9.24 2.7c3.45-6.56-1.42-10.4-4.77-13.53-5.36-5.03-10.7-7.2-16.8-.23-6.1 6.98-2.24 15.07 3.35 19.06 5.58 4 14.78 1.25 18.22-5.3"/></svg></a>
                    </section>
                    <section class="navbar-section desktop-menu">

                        <nav class="dropmenu animated">
                                                    
<ul >
        
</ul>

                                                </nav>

                        
                    </section>
                </nav>
            </section>
        </section>
        <div class="mobile-menu">
            <div class="button_container" id="toggle">
                <span class="top"></span>
                <span class="middle"></span>
                <span class="bottom"></span>
            </div>
        </div>
    
    
        <section id="start">
                    <section id="body-wrapper" class="section">
                <section class="container grid-lg">
                                            
                                            <h3>Long-term support projects</h3>
<ul>
<li>Active projects</li>
<li>Past projects</li>
</ul>
<p>Click on a project for further details and links to publications, analysis tools and data.</p>
<p>_unfold<em>more</em> Show all details _unfold<em>less</em> Hide all details</p>
<ul>
<li>1000 ancient genomes  PI: Anders Götherström (SU) </li>
</ul>
<p>Application round: May 2019 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Love Dalén (NRM) </p>
<ul>
<li>A new diagnostic test for rheumatoid arthritis  PI: Rikard Holmdahl (KI) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Inger Gjertsson (GU) </p>
<ul>
<li>Acquisition of lymphatic endothelial cell fate during zebrafish development  PI: Kaska Koltowska (UU) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Co-applicants: Hannah Stanforth (UU) </p>
<ul>
<li>Adaptive deregulation of miRNA genes by genomic rearrangements  PI: Carlos Rovira (LU) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>B cell receptor diversity and convergence after COVID-19 vaccination  PI: Karin Loré (KI) </li>
</ul>
<p>Application round: Jan 2021 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Baltic sea CRISPR dynamics  PI: Anders Andersson (KTH) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: Biodiversity</p>
<ul>
<li>Barcode-linked reads technology for genome sequencing and haplotyping  PI: Afshin Ahmadian (KTH) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: New Technology Development</p>
<ul>
<li>Cellular heterogeneity and differentiation during salamander regeneration  PI: Andras Simon (KI) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Chromatin dynamics and inheritance  PI: Simon Elsässer (KI) </li>
</ul>
<p>Application round: May 2017 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Chromatin-associated RNAs and their roles in chromatin regulation  PI: Neus Visa (SU) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Co-applicants: Jordi Planells (SU), Vincent Pelechano (KI) </p>
<ul>
<li>Clonal evolution from diagnosis to relapse in acute lymphoblastic leukemia  PI: Ann-Christine Syvänen (UU) </li>
</ul>
<p>Application round: Oct 2016 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>Correlation between patient-derived CAR-T cells and treatment outcome  PI: Magnus Essand (UU) </li>
</ul>
<p>Application round: Jan 2021 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Data integration for molecular classification of multiple sclerosis patients  PI: Maja Jagodic (KI) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1073/pnas.2011574118&gt;"> Zheleznyakova <em>et al.</em> <strong>Small noncoding RNA profiling across cellular and biofluid compartments and their implications for multiple sclerosis immunopathology.</strong> <em>Proc Natl Acad Sci USA</em> 118:e2011574118 (2021) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://doi.org/10.5878/c1mq-9r62&gt;">ncRNA counts</a></p>
<ul>
<li>Decoding the birth of hair follicles by single-cell RNA-seq  PI: Maria Kasper (KI) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Defining cell heterogeneity and therapy resistance in neuroblastoma  PI: Per Kogner (KI) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Cancer Biology</p>
<p>Co-applicants: Ninib Baryawno (KI) </p>
<ul>
<li>Delineating the genetics in severe scoliosis  PI: Paul Gerdhem (KI) </li>
</ul>
<p>Application round: Oct 2020 </p>
<p>Subject: Human Genetics</p>
<p>Co-applicants: Elisabet Einarsdottir (KTH), Juha Kere (KI) </p>
<ul>
<li>Dissection of cell-fate decisions in early B lymphocyte activation  PI: Taras Kreslavskiy (KI) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Environmental genomics in the Northern krill  PI: Andreas Wallberg (UU) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Epigenetic regulation of human skin wound healing  PI: Ning Xu Landén (KI) </li>
</ul>
<p>Application round: Oct 2020 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Epigenetics of acute myeloid leukemia  PI: Karl Ekwall (KI) </li>
</ul>
<p>Application round: Nov 2014 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>Establishment of epigenomic landscapes in the Drosophila embryo  PI: Mattias Mannervik (SU) </li>
</ul>
<p>Application round: May 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Evolution of human skeletal muscle metabolism  PI: Ola Hansson (LU) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Evolution of sex chromosomes  PI: Bengt Hansson (LU) </li>
</ul>
<p>Application round: Oct 2016 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Evolutionary genomics of Darwin&amp;aposs finches  PI: Leif Andersson (UU) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Mats Pettersson (UU) </p>
<ul>
<li>Evolutionary genomics of the Motacilla (wagtails) radiation  PI: Per Alström (UU) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Erik Enbody (UU), Leif Andersson (UU) </p>
<ul>
<li>Fat heterogeneity  PI: Kirsty Spalding (KI) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Fate mapping of human cells  PI: Jonas Frisén (KI) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s13059-019-1673-8&gt;"> Hård <em>et al.</em> <strong>Conbase: a software for unsupervised discovery of clonal somatic mutations in single cells through read phasing.</strong> <em>Genome Biol</em> 20:68 (2019) </a></p>
<ul>
<li>From single nuclei genomes to the arbuscular mycorrhizal tree of life  PI: Anna Rosling Larsson (UU) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Genome and metagenome of the deep-sea Swedish sponge Geodia barretti  PI: Paco Cárdenas (UU) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Biodiversity</p>
<ul>
<li>Genome-wide consequences of population declines in endangered species  PI: Love Dalén (NRM) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Johanna von Seth (SU), Nicolas Dussex (NRM) </p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1098/rstb.2019.0212&gt;"> Larsson <em>et al.</em> <strong>Consequences of past climate change and recent human persecution on mitogenomic diversity in the arctic fox.</strong> <em>Philos Trans R Soc Lond B Biol Sci</em> 374:20190212 (2019) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.cub.2020.07.046&gt;"> Lord <em>et al.</em> <strong>Pre-extinction Demographic Stability and Genomic Signatures of Adaptation in the Woolly Rhinoceros.</strong> <em>Curr Biol</em> 30:3871-3879.e7 (2020) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-021-22386-8&gt;"> von Seth <em>et al.</em> <strong>Genomic insights into the conservation status of the world &amp;aposs last remaining Sumatran rhinoceros populations.</strong> <em>Nat Commun</em> 12:2393 (2021) </a></p>
<ul>
<li>Genome-wide screen for inhibitors of lysosomal damage-induced cell death  PI: Sonja Aits (LU) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Genomic analysis of adaptation to climate change in two bumblebee species  PI: Matthew Webster (UU) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/molbev/msab086&gt;"> Christmas <em>et al.</em> <strong>Genetic barriers to historical gene flow between cryptic species of alpine bumblebees revealed by comparative population genomics.</strong> <em>Mol Biol Evol</em> advance online publication (2021) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP284846%3E">WGS reads</a></p>
<ul>
<li>Genomic analysis of pediatric solid tumors  PI: Monica Nistér (KI) </li>
</ul>
<p>Application round: May 2017 </p>
<p>Subject: Cancer Biology</p>
<p>Co-applicants: Teresita Diaz de Ståhl (KI), Johanna Sandgren (KI), Maxime Garcia (KI), Gustaf Ljungman (UU) </p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.12688/f1000research.16665.1&gt;"> Garcia <em>et al.</em> <strong>Sarek: A portable workflow for whole-genome sequencing analysis of germline and somatic variants.</strong> <em>F1000Res</em> 9:63 (2020) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;http://opensource.scilifelab.se/projects/sarek/&gt;"> <strong>Sarek</strong> – Complete open source pipeline to detect germline or somatic variants from normal or tumour/normal whole-genome sequencing data. </a></p>
<ul>
<li>Genomic and transcriptome profiling of MDS cells  PI: Eva Hellström-Lindberg (KI) </li>
</ul>
<p>Application round: May 2019 </p>
<p>Subject: Cancer Biology</p>
<p>Co-applicants: Seishi Ogawa (Kyoto Univ. and KI) </p>
<ul>
<li>Genomic dynamics over space and time in salmonid metapopulations  PI: Linda Laikre (SU) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1002/ece3.5646&gt;"> Kurland <em>et al.</em> <strong>Exploring a Pool‐seq‐only approach for gaining population genomic insights in nonmodel species.</strong> <em>Ecol Evol</em> 9:11448-11463 (2019) </a></p>
<ul>
<li>Identification of functionally perturbing non-coding mutations  PI: Robert Måsson (KI) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>Impact of spaceflight on mouse heart spatial gene expression  PI: Stefania Giacomello (KTH) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Improved gene-finding annotation in complex genomes  PI: Anders Blomberg (GU) </li>
</ul>
<p>Application round: May 2019 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Imputing genetic variants in a high kinship human cohort  PI: Åsa Johansson (UU) </li>
</ul>
<p>Application round: Jun 2015 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41598-019-53111-7&gt;"> Höglund <em>et al.</em> <strong>Improved power and precision with whole genome sequencing data in genome-wide association studies of inflammatory biomarkers.</strong> <em>Sci Rep</em> 9:16844 (2019) </a></p>
<ul>
<li>Insect aging omics  PI: Christopher Wheat (SU) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1242/jeb.169508&gt;"> Lehmann <em>et al.</em> <strong>Metabolome dynamics of diapause in the butterfly Pieris napi: distinguishing maintenance, termination and post-diapause phases.</strong> <em>J Exp Biol</em> 221:jeb169508 (2018) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://datadryad.org/resource/doi:10.5061/dryad.f0dg5&gt;">Cold tolerance and metabolome data</a></p>
<ul>
<li>Integrate multiple omics in 9000 Swedish individuals  PI: Marju Orho-Melander (LU) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Louise Brunkwall (LU) </p>
<ul>
<li>Interneuronal populations in locomotor networks  PI: Ole Kiehn (KI) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Intragenomic conflict  PI: Hanna Johannesson (UU) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1080/15592294.2020.1741758&gt;"> Hosseini <em>et al.</em> <strong>Comparative analysis of genome-wide DNA methylation in Neurospora.</strong> <em>Epigenetics</em> 15:972-987 (2020) </a></p>
<ul>
<li>Is DNA methylation heritable in clonal water fleas?  PI: Tobias Uller (LU) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Reinder Radersma (LU) </p>
<ul>
<li>Long-term evolution of germline-restricted chromosomes in songbirds  PI: Alexander Suh (UU) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Francisco Ruiz-Ruano (UU) </p>
<ul>
<li>Longitudinal proteomic profiling in blood and airways of COVID-19 patients  PI: Anna Smed Sörensen (KI) </li>
</ul>
<p>Application round: Oct 2020 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Machine learning approach to spatially resolved sequencing in brain tissue  PI: Tomas Björklund (LU) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: New Technology Development</p>
<ul>
<li>Massive and parallel single-cell sequencing of eukaryotic plankton  PI: Anders Andersson (KTH) </li>
</ul>
<p>Application round: Oct 2020 </p>
<p>Subject: New Technology Development</p>
<p>Co-applicants: Rachel Foster (SU) </p>
<ul>
<li>Microbes in the deep  PI: Mark Dopson (LNU) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: Biodiversity</p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/nbis-meta&gt;"> <strong>Metagenomics pipeline</strong> – a Snakemake workflow for processing and analysis of metagenomic sequence reads </a></p>
<ul>
<li>Mitochondrial dysfunction in Parkinson's disease  PI: Nils-Göran Larsson (KI) </li>
</ul>
<p>Application round: May 2017 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Molecular pathogenesis of liver lipotoxicity  PI: Margit Mahlapuu (GU) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Multiomics analysis of response to epigenetic treatments in multiple myeloma  PI: Helena Jernberg Wiklund (UU) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Cancer Biology</p>
<p>Co-applicants: Jan Komorowski (UU) </p>
<ul>
<li>Mutation calling to enable sensitive proteogenomics for cancer immunotherapy  PI: Janne Lehtiö (KI) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: New Technology Development</p>
<p>Co-applicants: Rui Branca (KI) </p>
<ul>
<li>Non-coding regulatory RNAs in the transition from uni- to multicellularity  PI: Fredrik Söderbom (UU) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Novel regulatory patterns of biofilm formation in Candida parapsilosis  PI: Ute Römling (KI) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Omics of sleep loss  PI: Christian Benedict (UU) </li>
</ul>
<p>Application round: Oct 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Co-applicants: Jonathan Cedernaes (UU) </p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1126/sciadv.aar8590&gt;"> Cedernaes <em>et al.</em> <strong>Acute sleep loss results in tissue-specific alterations in genome-wide DNA methylation state and metabolic fuel utilization in humans.</strong> <em>Sci Adv</em> 4:eaar8590 (2018) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/orzechoj/cedernaes2018_analysis/tree/master&gt;"> <strong>RNA-seq and DNA methylation analysis</strong> – R scripts and output </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6903/&gt;">RNA-seq data</a> <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6908/&gt;">DNA methylation data</a></p>
<ul>
<li>Oral microbiota and immune dynamics in early cancer protection  PI: Margaret Sällberg Chen (KI) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1021/acs.jproteome.1c00018&gt;"> Morgell <em>et al.</em> <strong>Metabolic characterization of plasma and cyst fluid from cystic precursors to pancreatic cancer patients reveal metabolic signatures of bacterial infection.</strong> <em>J Proteome Res</em> 20:2725-2738 (2021) </a></p>
<ul>
<li>Organ-specific features of the lymphatic vasculature in the gastrointestinal tract  PI: Taija Mäkinen (UU) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Pericyte fibrosis  PI: Christian Göritz (KI) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop&gt;"> <strong>Single-cell RNA-seq pipeline</strong> – a Snakemake workflow for quality control and basic analysis of single-cell RNA-seq data (under development) </a></p>
<ul>
<li>Pneumonial metagenomics, metatranscriptomics and metabolomics  PI: Birgitta Henriques-Normark (KI) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/nbis-meta&gt;"> <strong>Metagenomics pipeline</strong> – a Snakemake workflow for processing and analysis of metagenomic sequence reads </a></p>
<ul>
<li>Preimplantation stress: programming our future generations  PI: Sophie Petropoulos (KI) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Profiling of drug treatments on sarcoma and coronavirus-infected cell lines  PI: Ola Spjuth (UU) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Sven Nelander (UU), Jordi Carreras-Puigvert (KI) </p>
<ul>
<li>Prophylactic probiotics to extremely low birth weight preterm infants  PI: Thomas Abrahamsson (LiU) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Maria Jenmalm (LiU) </p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/nbis-meta&gt;"> <strong>Metagenomics pipeline</strong> – a Snakemake workflow for processing and analysis of metagenomic sequence reads </a></p>
<ul>
<li>Proteogenomics of radiotherapy resistance in locally recurring breast cancer  PI: Emma Niméus (LU) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Cancer Biology</p>
<p>Co-applicants: Lars Malmström (Univ. of Zurich) </p>
<ul>
<li>Quantifying the dynamics of the blood plasma proteome during acute HIV  PI: Joakim Esbjörnsson (LU) </li>
</ul>
<p>Application round: Jan 2021 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Jamirah Nazziwa (LU) </p>
<ul>
<li>Redefining the molecular landscape of triple-negative breast cancer  PI: Johan Staaf (LU) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>Reproductive isolation and the evolution of key innovations  PI: Charlie Cornwallis (LU) </li>
</ul>
<p>Application round: Feb 2015 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Respiratory genomics in flies and mammals  PI: Christos Samakovlis (SU) </li>
</ul>
<p>Application round: Nov 2014 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Seqitope - a novel tool for mapping antibody epitopes  PI: Mats Persson (KI) </li>
</ul>
<p>Application round: Oct 2020 </p>
<p>Subject: New Technology Development</p>
<p>Co-applicants: Anders Andersson (KTH) </p>
<ul>
<li>Sex-biased gene expression and its dependence on condition  PI: Urban Friberg (LiU) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Sex-limited experimental evolution of novel sex chromosomes  PI: Jessica Abbott (LU) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Single cell profiling of cell types and lineages in the mouse brain  PI: Jonas Frisén (KI) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: New Technology Development</p>
<ul>
<li>Single-cell analysis of SARS-CoV-2 infection in a 3D primary cell lung  PI: Johan Henriksson (UmU) </li>
</ul>
<p>Application round: Jan 2021 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Anna Överby Wernstedt (UmU), Annasara Lenman (UmU) </p>
<ul>
<li>Single-cell gene expression analysis of limb regeneration  PI: Andras Simon (KI) </li>
</ul>
<p>Application round: Jun 2015 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Somatic mutations and its implications for age-related disease  PI: Maria Eriksson (KI) </li>
</ul>
<p>Application round: Oct 2020 </p>
<p>Subject: Human Genetics</p>
<ul>
<li>SuperGenE - supergene evolution in a classic plant system  PI: Tanja Slotte (SU) </li>
</ul>
<p>Application round: Feb 2019 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Symbiont-induced host speciation in insects  PI: Lisa Klasson (UU) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Systems biology dynamics of liver regeneration in humans  PI: Niklas Björkström (KI) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Co-applicants: Martin Cornillet (KI) </p>
<ul>
<li>Targeting topoisomerase regulation as a mean to control transcription in cancer cells  PI: Laura Baranello (KI) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>The basis of inherited migratory directions in the Australian bogong moth  PI: Eric Warrant (LU) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>The epigenetic landscape in MLL-rearranged leukemia  PI: Anna Andersson (LU) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>The genomics and transcriptomics behind Nature&amp;aposs high performance material  PI: Anna Rising (KI) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>The molecular basis of recombination rate variation in butterflies  PI: Niclas Backström (UU) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>The role for gene repression in maintaining differentiated identity  PI: Johan Holmberg (KI) </li>
</ul>
<p>Application round: May 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>The transcriptional landscape of mucosal healing  PI: Eduardo Villablanca (KI) </li>
</ul>
<p>Application round: May 2020 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Towards tailored treatments in multiple sclerosis  PI: Joachim Burman (UU) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Kim Kultima (UU) </p>
<ul>
<li>Tracing human history in southern Africa using ancient proteomics  PI: Carina Schlebusch (UU) </li>
</ul>
<p>Application round: Jan 2021 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Co-applicants: Imke Lankheet (UU) </p>
<ul>
<li>Transcriptional and immune receptor profiling of influenza-specific B cells  PI: Davide Angeletti (GU) </li>
</ul>
<p>Application round: Oct 2019 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.celrep.2021.109286&gt;"> Matthew <em>et al.</em> <strong>Single-cell BCR and transcriptome analysis after influenza infection reveals spatiotemporal dynamics of antigen-specific B cells.</strong> <em>Cell Rep</em> 35:109286 (2021) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/angelettilab/scMouseBcellFlu&gt;"> <strong>Data analysis scripts</strong> – Code and supporting files for reproducing the analyses reported in the publication </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9478/&gt;">VDJ data</a> <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9491/&gt;">Gene expression data</a></p>
<ul>
<li>Transcriptional profiling of mononuclear phagocytes in sarcoidosis patients  PI: Anna Smed Sörensen (KI) </li>
</ul>
<p>Application round: May 2018 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1183/13993003.03468-2020&gt;"> Lepzien <em>et al.</em> <strong>Monocytes in sarcoidosis are potent TNF producers and predict disease outcome.</strong> <em>Eur Respir J</em> 2003468 (2021) </a></p>
<ul>
<li>Transcriptome analysis of memory T cells during influenza virus challenge  PI: Nils Lycke (GU) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Transcriptome analysis of the response to mechanical stimulation in cereals  PI: Olivier van Aken (LU) </li>
</ul>
<p>Application round: Jan 2020 </p>
<p>Subject: Biodiversity</p>
<p>Co-applicants: Nick Sirijovski (LU), Essam Darwish (LU) </p>
<ul>
<li>Understanding epigenetics of type 2 diabetes by WGBS  PI: Charlotte Ling (LU) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Alexander Perfilyev (LU) </p>
<ul>
<li>Understanding the tissue-material interface during wound healing  PI: Molly Stevens (KI) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Alessondra Speidel (KI), Annina Steinbach (KI) </p>
<ul>
<li>WGS and RNA-seq in chronic lymphocytic leukemia subsets  PI: Richard Rosenquist (UU) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Cancer Biology</p>
<ul>
<li>Whole genome sequencing to identify genetic causes of adverse drug reactions  PI: Mia Wadelius (UU) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Human Genetics</p>
<ul>
<li>Whole-genome sequencing of 1000 colorectal cancers  PI: Tobias Sjöblom (UU) </li>
</ul>
<p>Application round: Oct 2018 </p>
<p>Subject: Cancer Biology</p>
<p>Co-applicants: Mathias Uhlén (KTH) </p>
<ul>
<li>Why such high genetic diversity in songbird immunity?  PI: Helena Westerdahl (LU) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>scRNA-seq analysis of B cells in patients with Sjögren&amp;aposs Syndrome  PI: Ann-Christine Syvänen (UU) </li>
</ul>
<p>Application round: Jan 2021 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Co-applicants: Gunnel Nordmark (UU), Gustav Arvidsson (UU), Amanda Raine (UU) </p>
<pre><code>  Organization,No of projects
  GU, 4
  KI, 36
  KTH, 4
  LNU, 1
  LU, 17
  LiU, 2
  NRM, 1
  SU, 7
  UU, 25
  UmU, 1</code></pre>
<p>Click on a project for further details and links to publications, analysis tools and data.</p>
<p>_unfold<em>more</em> Show all details _unfold<em>less</em> Hide all details</p>
<ul>
<li>Acoelomorph genomes  PI: Ulf Jondelius (NRM) </li>
</ul>
<p>Application round: Jun 2015 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Analysis of the nuclear and cytosolic transcriptomes in human brain  PI: Lars Feuk (UU) </li>
</ul>
<p>Application round: Feb 2015 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41598-018-35001-6&gt;"> Zaghlool <em>et al.</em> <strong>Expression profiling and in situ screening of circular RNAs in human tissues.</strong> <em>Sci Rep</em> 8:16953 (2018) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41598-021-83541-1&gt;"> Zaghlool <em>et al.</em> <strong>Characterization of the nuclear and cytosolic transcriptomes in human brain tissue reveals new insights into the subcellular distribution of RNA transcripts.</strong> <em>Sci Rep</em> 11:4076 (2021) </a></p>
<ul>
<li>Ancient DNA metagenome  PI: Barbara Wohlfarth (SU) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Biodiversity</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.quascirev.2017.11.037&gt;"> Ahmed <em>et al.</em> <strong>Archaeal community changes in lateglacial lake sediments: Evidence from ancient DNA.</strong> <em>Quat Sci Rev</em> 181:19-29 (2018) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.3389/fevo.2019.00189&gt;"> Parducci <em>et al.</em> <strong>Shotgun environmental DNA, pollen, and macrofossil analysis of lateglacial lake sediments from southern Sweden.</strong> <em>Front Ecol Evol</em> 7:189 (2019) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/ena/data/view/PRJNA378719&gt;">DNA sequence reads</a></p>
<ul>
<li>Antibody repertoires  PI: Gunilla Karlsson Hedestam (KI) </li>
</ul>
<p>Application round: May 2014 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/ncomms13642&gt;"> Corcoran <em>et al.</em> <strong>Production of individualized V gene databases reveals high levels of immunoglobulin genetic diversity.</strong> <em>Nat Commun</em> 7:13642 (2016) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;http://docs.igdiscover.se/&gt;"> <strong>IgDiscover</strong> – Software for antibody repertoire analysis and V gene discovery from high-throughput sequencing reads </a></p>
<ul>
<li>Bacterial regulatory RNAs  PI: Emmanuelle Charpentier (UmU) </li>
</ul>
<p>Application round: Oct 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1080/15476286.2015.1110674&gt;"> Le Rhun <em>et al.</em> <strong>RNA sequencing uncovers antisense RNAs and novel small RNAs in <em>Streptococcus pyogenes</em>.</strong> <em>RNA Biol</em> 13:177-195 (2015) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/nar/gkw1316&gt;"> Le Rhun <em>et al.</em> <strong>Identification of endoribonuclease specific cleavage positions reveals novel targets of RNase III in <em>Streptococcus pyogenes</em>.</strong> <em>Nucleic Acids Res</em> gkw1316 (2017) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/bioproject/PRJNA341673&gt;">RNA-seq data set 1</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/bioproject/PRJNA341675&gt;">RNA-seq data set 2</a></p>
<ul>
<li>Beetle genome assembly and annotation  PI: Göran Arnqvist (UU) </li>
</ul>
<p>Application round: May 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/gbe/evx205&gt;"> Sayadi <em>et al.</em> <strong>The evolution of dark matter in the mitogenome of seed beetles.</strong> <em>Genome Biol Evol</em> 9:2697-2706 (2017) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41559-019-1041-9&gt;"> Sayadi <em>et al.</em> <strong>The genomic footprint of sexual conflict.</strong> <em>Nat Ecol Evol</em> 3:1725-1730 (2019) </a></p>
<ul>
<li>Biodiversity in air  PI: Per Stenberg (UmU) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Biodiversity</p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/nbis-meta&gt;"> <strong>Metagenomics pipeline</strong> – a snakemake workflow for processing and analysis of metagenomic sequence reads </a></p>
<ul>
<li>Body weight sensor  PI: Claes Ohlsson (GU) </li>
</ul>
<p>Application round: Oct 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Cancer proteogenomics  PI: Janne Lehtiö (KI) </li>
</ul>
<p>Application round: Nov 2014 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-018-03311-y&gt;"> Zhu <em>et al.</em> <strong>Discovery of coding regions in the human genome by integrated proteogenomics analysis workflow.</strong> <em>Nat Commun</em> 9:903 (2018) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-019-09018-y&gt;"> Johansson <em>et al.</em> <strong>Breast cancer quantitative proteome and proteogenomic landscape.</strong> <em>Nat Commun</em> 10:1600 (2019) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD006291%3E">Proteomics data for A431 cells and tissues</a> <a href="/support/ltsprojects/&lt;http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD008841%3E">Proteomics data for breast tumors</a> <a href="/support/ltsprojects/&lt;http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD011385%3E">Proteomics data for tumor and matched normal tissue</a> <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5285&gt;">RNA-seq data for A431 cells</a> <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2836&gt;">RNA-seq data for tissues</a></p>
<ul>
<li>Colon cancer genes  PI: Erik Johansson (UmU) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1371/journal.pone.0213350&gt;"> Rentoft <em>et al.</em> <strong>A geographically matched control population efficiently limits the number of candidate disease-causing variants in an unbiased whole-genome analysis.</strong> <em>PLoS One</em> 14:e0213350 (2019) </a></p>
<ul>
<li>DACP rat genome sequencing  PI: Ulrika Norin (KI) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-020-20586-2&gt;"> Norin <em>et al.</em> <strong>Endophilin A2 deficiency protects rodents from autoimmune arthritis by modulating T cell activation.</strong> <em>Nat Commun</em> 12:610 (2021) </a></p>
<ul>
<li>Development and diversity of midbrain dopamine neurons  PI: Thomas Perlmann (KI) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.stem.2016.10.003&gt;"> Kee <em>et al.</em> <strong>Single-cell analysis reveals a close relationship between differentiating dopamine and subthalamic nucleus neuronal lineages.</strong> <em>Cell Stem Cell</em> 20:29-40 (2017) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-019-08453-1&gt;"> Tiklová <em>et al.</em> <strong>Single-cell RNA sequencing reveals midbrain dopamine neuron diversity emerging during mouse brain development.</strong> <em>Nat Commun</em> 10:581 (2019) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87069%3E">RNA-seq data set 1</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE116138%3E">RNA-seq data set 2</a></p>
<ul>
<li>Diabetogenic perturbations of the fetal methylome  PI: Paul Franks (LU) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: New Technology Development</p>
<ul>
<li>Diseases in human history  PI: Kristian Kristiansen (GU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Biodiversity</p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/nbis-meta&gt;"> <strong>Metagenomics pipeline</strong> – a snakemake workflow for processing and analysis of metagenomic sequence reads </a></p>
<ul>
<li>Endurance training and gene expression in human muscle  PI: Carl-Johan Sundberg (KI) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1096/fj.14-255000&gt;"> Lindholm <em>et al.</em> <strong>The human skeletal muscle transcriptome: sex differences, alternative splicing, and tissue homogeneity assessed with RNA sequencing.</strong> <em>FASEB J</em> 28:4571-4581 (2014) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s12864-015-1686-y&gt;"> Lindskog <em>et al.</em> <strong>The human cardiac and skeletal muscle proteomes defined by transcriptomics and antibody-based profiling.</strong> <em>BMC Genomics</em> 16:475 (2015) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1371/journal.pgen.1006294&gt;"> Lindholm <em>et al.</em> <strong>The impact of endurance training on human skeletal muscle memory, global isoform expression and novel transcripts.</strong> <em>PLoS Genet</em> 12:e1006294 (2016) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE58387%3E">RNA-seq data set 1</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE58608%3E">RNA-seq data set 2</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60590%3E">RNA-seq data set 3</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60833%3E">RNA-seq data set 4</a></p>
<ul>
<li>Epigenetic states of oligodendrocyte lineage cells  PI: Gonçalo Castelo-Branco (KI) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.devcel.2018.07.005&gt;"> Marques <em>et al.</em> <strong>Transcriptional convergence of oligodendrocyte lineage progenitors during development.</strong> <em>Dev Cell</em> 46:504-517 (2018) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41591-018-0236-y&gt;"> Falcão <em>et al.</em> <strong>Disease-specific oligodendrocyte lineage cells arise in multiple sclerosis.</strong> <em>Nat Med</em> 24:1837-1844 (2018) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://castelobranco.shinyapps.io/OPCsinglecell2017/&gt;"> <strong>OPCs throughout mouse development</strong> – Web resource for exploring gene expression data from Marques et al. </a> <a href="/support/ltsprojects/&lt;https://castelobranco.shinyapps.io/EAE2017/&gt;"> <strong>Oligodendrocyte lineage in a mouse model of multiple sclerosis</strong> – Web resource for exploring gene expression data from Falcão et al. </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95093%3E">Bulk RNA-seq data</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95194%3E">Single-cell RNA-seq data 1</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113973%3E">Single-cell RNA-seq data 2</a></p>
<ul>
<li>Evolution of gene regulation in plants  PI: Tanja Slotte (UU) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/molbev/msv169&gt;"> Steige <em>et al.</em> <strong><em>Cis</em> -regulatory changes associated with a recent mating system shift and floral adaptation in <em>Capsella</em>.</strong> <em>Mol Biol Evol</em> 32:2501-2514 (2015) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1101/044016&gt;"> Steige <em>et al.</em> <strong>The role of transposable elements for gene expression in <em>Capsella</em> hybrids and allopolyploids.</strong> <em>bioRxiv</em> 44016 (2016) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1073/pnas.1612561114&gt;"> Steige <em>et al.</em> <strong>Genomic analysis reveals major determinants of <em>cis</em> -regulatory variation in <em>Capsella grandiflora</em>.</strong> <em>Proc Natl Acad Sci USA</em> 114:1087-1092 (2017) </a></p>
<ul>
<li>Examination of embryonic and adult stem cell characteristics  PI: Jonas Muhr (KI) </li>
</ul>
<p>Application round: May 2014 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1101/gr.203513.115&gt;"> Hagey <em>et al.</em> <strong>Distinct transcription factor complexes act on a permissive chromatin landscape to establish regionalized gene expression in CNS stem cells.</strong> <em>Genome Res</em> 26:908-917 (2016) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.15252/embr.201846635&gt;"> Klum <em>et al.</em> <strong>Sequentially acting SOX proteins orchestrate astrocyte‐ and oligodendrocyte‐specific gene expression.</strong> <em>EMBO Rep</em> e46635 (2018) </a></p>
<ul>
<li>Exercise-regulated mechanisms in mouse organs  PI: Pontus Boström (KI) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Exploring sex-biased gene expression in willows  PI: Sofia Berlin Kolm (SLU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s12915-020-00808-1&gt;"> Almeida <em>et al.</em> <strong>Genome assembly of the basket willow, Salix viminalis, reveals earliest stages of sex chromosome expansion.</strong> <em>BMC Biol</em> 18:78 (2020) </a></p>
<ul>
<li>Finding causal variants within schizophrenia risk loci  PI: Patrick Sullivan (KI) </li>
</ul>
<p>Application round: Oct 2016 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-020-15707-w&gt;"> Halvorsen <em>et al.</em> <strong>Increased burden of ultra-rare structural variants localizing to boundaries of topologically associated domains in schizophrenia.</strong> <em>Nat Commun</em> 11:1842 (2020) </a></p>
<ul>
<li>Finding new treatments for type 2 diabetes  PI: Nils Wierup (LU) </li>
</ul>
<p>Application round: Feb 2015 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Flycatcher genomics  PI: Hans Ellegren (UU) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1101/gr.196485.115&gt;"> Burri <em>et al.</em> <strong>Linked selection and recombination rate variation drive the evolution of the genomic landscape of differentiation across the speciation continuum of <em>Ficedula</em> flycatchers.</strong> <em>Genome Res</em> 25:1656-1665 (2015) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;http://www.ebi.ac.uk/ena/data/view/PRJNA75089&gt;">Reference genome assembly</a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.5061/dryad.v0v83&gt;">Genotypes for linkage analysis</a> <a href="/support/ltsprojects/&lt;http://www.ebi.ac.uk/ena/data/view/PRJEB7359&gt;">Resequencing data</a></p>
<ul>
<li>Fungal genome annotation through proteogenomics  PI: Annika Scheynius (KI) </li>
</ul>
<p>Application round: May 2013 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/nar/gkx006&gt;"> Zhu <em>et al.</em> <strong>Proteogenomics produces comprehensive and highly accurate protein-coding gene annotation in a complete genome assembly of <em>Malassezia sympodialis</em>.</strong> <em>Nucleic Acids Res</em> 45:2629-2643 (2017) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/ena/data/view/PRJEB13283&gt;">DNA sequence reads, genome assemblies and gene annotation</a> <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-4589&gt;">RNA-seq data</a> <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/pride/archive/projects/PXD003773&gt;">Proteomics data</a></p>
<ul>
<li>Gene regulatory networks in β-cells  PI: Olov Andersson (KI) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s42255-021-00391-x&gt;"> Charbord <em>et al.</em> <strong>In vivo screen identifies a SIK inhibitor that induces β cell proliferation through a transient UPR.</strong> <em>Nat Metab</em> 3:682-700 (2021) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;http://rshiny.nbis.se/shiny-server-apps/o_andersson/&gt;"> <strong>Data exploration</strong> – Web resource for exploring gene expression data from Charbord et al. </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE163972%3E">RNA-seq data</a></p>
<ul>
<li>Genetic associations of serum lipidome in a prospective cohort  PI: Olle Melander (LU) </li>
</ul>
<p>Application round: May 2014 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1161/jaha.116.004503&gt;"> Kjellqvist <em>et al.</em> <strong>Identification of shared and unique serum lipid profiles in diabetes mellitus and myocardial infarction.</strong> <em>J Am Heart Assoc</em> 5:e004503 (2016) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.2337/dc19-1199&gt;"> Fernandez <em>et al.</em> <strong>Plasma lipidome and prediction of type 2 diabetes in the population-based Malmö diet and cancer cohort.</strong> <em>Dia Care</em> 43:366-373 (2020) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.ijcard.2021.01.059&gt;"> Ottosson <em>et al.</em> <strong>A plasma lipid signature predicts incident coronary artery disease.</strong> <em>Int J Cardiol</em> 331:249-254 (2021) </a></p>
<ul>
<li>Genetic risk factors for ANCA-associated vasculitides  PI: Kerstin Lindblad-Toh (UU) </li>
</ul>
<p>Application round: Feb 2015 </p>
<p>Subject: Human Genetics</p>
<ul>
<li>Genetic variation of European aspen  PI: Pär Ingvarsson (UmU) </li>
</ul>
<p>Application round: Oct 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s13059-018-1444-y&gt;"> Wang <em>et al.</em> <strong>A major locus controls local adaptation and adaptive life history variation in a perennial plant.</strong> <em>Genome Biol</em> 19:72 (2018) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1073/pnas.1801437115&gt;"> Lin <em>et al.</em> <strong>Functional and evolutionary genomic inferences in Populus through genome and population sequencing of American and European aspen.</strong> <em>Proc Natl Acad Sci USA</em> 115:E10970-E10978 (2018) </a></p>
<ul>
<li>Genetics of heart stress recovery  PI: Jan Borén (GU) </li>
</ul>
<p>Application round: Oct 2013 </p>
<p>Subject: Human Genetics</p>
<ul>
<li>Genome-wide epigenetic variation in type 2 diabetes  PI: Charlotte Ling (LU) </li>
</ul>
<p>Application round: Nov 2014 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41598-019-44076-8&gt;"> Bysani <em>et al.</em> <strong>ATAC-seq reveals alterations in open chromatin in pancreatic islets from subjects with type 2 diabetes.</strong> <em>Sci Rep</em> 9:7785 (2019) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50398%3E">Gene expression data</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129383%3E">ATAC-seq data</a></p>
<ul>
<li>Hemostatic regulation in human liver  PI: Christina Jern (GU) </li>
</ul>
<p>Application round: May 2014 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1055/s-0039-1687879&gt;"> Olsson Lindvall <em>et al.</em> <strong>Hemostatic genes exhibit a high degree of allele-specific regulation in liver.</strong> <em>Thromb Haemost</em> 119:1072-1083 (2019) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1055/s-0039-3401824&gt;"> Olsson Lindvall <em>et al.</em> <strong>A comprehensive sequencing-based analysis of allelic methylation patterns in hemostatic genes in human liver.</strong> <em>Thromb Haemost</em> 120:229-242 (2020) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1055/s-0040-1720980&gt;"> Olsson Lindvall <em>et al.</em> <strong>Comparison of DNA Methylation Profiles of Hemostatic Genes between Liver Tissue and Peripheral Blood within Individuals.</strong> <em>Thromb Haemost</em> 121:573-583 (2021) </a></p>
<ul>
<li>Human structural genetic variation  PI: Magnus Nordenskjöld (KI) </li>
</ul>
<p>Application round: Nov 2014 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.12688/f1000research.11168.2&gt;"> Eisfeldt <em>et al.</em> <strong>TIDDIT, an efficient and comprehensive structural variant caller for massive parallel sequencing data.</strong> <em>F1000Res</em> 6:664 (2017) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s13073-019-0675-1&gt;"> Lindstrand <em>et al.</em> <strong>From cytogenetics to cytogenomics: whole-genome sequencing as a first-line test comprehensively captures the diverse spectrum of disease-causing genetic variation underlying intellectual disability.</strong> <em>Genome Med</em> 11:68 (2019) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/NBISweden/vcf2cytosure&gt;"> <strong>vcf2cytosure</strong> – Convert structural variants from VCF format to the format used in CytoSure Interpret </a></p>
<ul>
<li>Impacts of nitrogen fertilisation on forest belowground fungal communities  PI: Nathaniel Street (UmU) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: Biodiversity</p>
<p>Co-applicants: Vaughan Hurry (SLU) </p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1128/msystems.00884-20&gt;"> Schneider <em>et al.</em> <strong>Comparative Fungal Community Analyses Using Metatranscriptomics and Internal Transcribed Spacer Amplicon Sequencing from Norway Spruce.</strong> <em>mSystems</em> 6:e00884-20 (2021) </a></p>
<ul>
<li>Improving transcriptome reconstruction  PI: Olof Emanuelsson (KTH) </li>
</ul>
<p>Application round: May 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/nar/gkx087&gt;"> Reimegård <em>et al.</em> <strong>Genome-wide identification of physically clustered genes suggests chromatin-level co-regulation in male reproductive development in <em>Arabidopsis thaliana</em>.</strong> <em>Nucleic Acids Res</em> 45:3253-3265 (2017) </a></p>
<ul>
<li>Integrated multiomics analysis of pediatric acute myeloid leukemia  PI: Linda Holmfeldt (UU) </li>
</ul>
<p>Application round: Jan 2018 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1182/bloodadvances.2020003709&gt;"> Stratmann <em>et al.</em> <strong>Genomic characterization of relapsed acute myeloid leukemia reveals novel putative therapeutic targets.</strong> <em>Blood Adv</em> 5:900-912 (2021) </a></p>
<ul>
<li>Iron metabolism and inflammation  PI: Anna-Lena Spetz (SU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Human Genetics</p>
<ul>
<li>Leptidea butterfly genomics  PI: Niclas Backström (UU) </li>
</ul>
<p>Application round: Feb 2015 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1111/mec.15188&gt;"> Talla <em>et al.</em> <strong>Lack of gene flow: Narrow and dispersed differentiation islands in a triplet of Leptidea butterfly species.</strong> <em>Mol Ecol</em> 28:3756-3770 (2019) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/ena/data/view/PRJEB21838&gt;">Leptidea DNA sequence reads, genome assembly and variant calls</a></p>
<ul>
<li>Marine genomics of barnacle  PI: Anders Blomberg (GU) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Biodiversity</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1007/s10126-021-10033-8&gt;"> Alm Rosenblad <em>et al.</em> <strong>Genomic characterization of the barnacle <em>Balanus improvisus</em> reveals extreme nucleotide diversity in coding regions.</strong> <em>Mar Biotechnol</em> advance online publication (2021) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://doi.org/10.17044/scilifelab.14339153&gt;">Pilot genome assembly</a></p>
<ul>
<li>Melanoma genetics  PI: Veronica Höiom (KI) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1002/gcc.22363&gt;"> Tuominen <em>et al.</em> <strong>The role of germline alterations in the DNA damage response genes <em>BRIP1</em> and <em>BRCA2</em> in melanoma susceptibility.</strong> <em>Genes Chromosomes Cancer</em> 55:601-611 (2016) </a></p>
<ul>
<li>Methods for multiplex protein analysis  PI: Ulf Landegren (UU) </li>
</ul>
<p>Application round: May 2017 </p>
<p>Subject: New Technology Development</p>
<p>Co-applicants: Caroline Gallant (UU) </p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s42003-021-02142-w&gt;"> Reimegård <em>et al.</em> <strong>A combined approach for single-cell mRNA and intracellular protein expression analysis.</strong> <em>Commun Biol</em> 4:624 (2021) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/b97jre/SPARC&gt;"> <strong>Scripts and data files</strong> – to generate figures and tables from the article </a></p>
<ul>
<li>Methylation QTL analysis in the chicken  PI: Dominic Wright (LiU) </li>
</ul>
<p>Application round: Jun 2015 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41559-020-01310-1&gt;"> Höglund <em>et al.</em> <strong>The methylation landscape and its role in domestication and gene regulation in the chicken.</strong> <em>Nat Ecol Evol</em> 4:1713-1724 (2020) </a></p>
<ul>
<li>MicroRNA in muscle dystrophy  PI: Madeleine Durbeej-Hjalt (LU) </li>
</ul>
<p>Application round: Nov 2014 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>MicroRNA networks in Parkinson's disease  PI: Johan Jakobsson (LU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Cell and Developmental Biology</p>
<ul>
<li>Multi-level omics profiling of the yeast cell cycle  PI: Jens Nielsen (CTH) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1073/pnas.1919535117&gt;"> Campbell <em>et al.</em> <strong>Building blocks are synthesized on demand during the yeast cell cycle.</strong> <em>Proc Natl Acad Sci USA</em> 117:7575-7583 (2020) </a></p>
<ul>
<li>Multi-omics analysis of human neuronal cells in health and disease  PI: Niklas Dahl (UU) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1007/s12035-019-1585-3&gt;"> Sobol <em>et al.</em> <strong>Transcriptome and proteome profiling of neural induced pluripotent stem cells from individuals with Down syndrome disclose dynamic dysregulations of key pathways and cellular functions.</strong> <em>Mol Neurobiol</em> 56:7113-7127 (2019) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.nbd.2019.104583&gt;"> Schuster <em>et al.</em> <strong>Transcriptomes of Dravet syndrome iPSC derived GABAergic cells reveal dysregulated pathways for chromatin remodeling and neurodevelopment.</strong> <em>Neurobiol Dis</em> 132:104583 (2019) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s13148-019-0803-1&gt;"> Laan <em>et al.</em> <strong>DNA methylation changes in Down syndrome derived neural iPSCs uncover co-dysregulation of ZNF and HOX3 families of transcription factors.</strong> <em>Clin Epigenetics</em> 12:9 (2020) </a></p>
<ul>
<li>Mutations in childhood acute lymphoblastic leukemia  PI: Ann-Christine Syvänen (UU) </li>
</ul>
<p>Application round: May 2013 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1002/humu.22719&gt;"> Lindqvist <em>et al.</em> <strong>The mutational landscape in pediatric acute lymphoblastic leukemia deciphered by whole genome sequencing.</strong> <em>Hum Mutat</em> 36:118-128 (2015) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.18632/oncotarget.11773&gt;"> Lindqvist <em>et al.</em> <strong>Deep targeted sequencing in pediatric acute lymphoblastic leukemia unveils distinct mutational patterns between genetic subtypes and novel relapse-associated genes.</strong> <em>Oncotarget</em> 7:64071-64088 (2016) </a></p>
<ul>
<li>Neo-antigens in melanoma  PI: Göran Jönsson (LU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-017-01460-0&gt;"> Lauss <em>et al.</em> <strong>Mutational and putative neoantigen load predict clinical benefit of adoptive T cell therapy in melanoma.</strong> <em>Nat Commun</em> 8:1738 (2017) </a></p>
<ul>
<li>Neuronal identity  PI: Johan Holmberg (KI) </li>
</ul>
<p>Application round: Jun 2015 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-018-03538-9&gt;"> Södersten <em>et al.</em> <strong>A comprehensive map coupling histone modifications with gene regulation in adult dopaminergic and serotonergic neurons.</strong> <em>Nat Commun</em> 9:1226 (2018) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107656%3E">RNA-seq, ChIP-seq and TRAP-seq data</a></p>
<ul>
<li>New mutations in dilated cardiomyopathy  PI: Martin Bergö (GU) </li>
</ul>
<p>Application round: May 2013 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.ijcard.2016.11.066&gt;"> Dalin <em>et al.</em> <strong>Massive parallel sequencing questions the pathogenic role of missense variants in dilated cardiomyopathy.</strong> <em>Int J Cardiol</em> 228:742-748 (2017) </a></p>
<ul>
<li>Next-gen plant phylogenetics  PI: Alexandre Antonelli (GU) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.7717/peerj.5175&gt;"> Andermann <em>et al.</em> <strong>SECAPR - a bioinformatics pipeline for the rapid and user-friendly processing of targeted enriched Illumina sequences, from raw reads to alignments.</strong> <em>PeerJ</em> 6:e5175 (2018) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/AntonelliLab/seqcap_processor&gt;"> <strong>SECAPR</strong> – pipeline for processing of sequence capture data </a></p>
<ul>
<li>Novel subpopulations in the brain reward-aversion system  PI: Åsa Mackenzie (UU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s42003-020-1028-8&gt;"> Wallén-Mackenzie <em>et al.</em> <strong>Spatio-molecular domains identified in the mouse subthalamic nucleus and neighboring glutamatergic and GABAergic brain structures.</strong> <em>Commun Biol</em> 3:338 (2020) </a></p>
<ul>
<li>Omics interplay in cardiometabolic disease  PI: Marju Orho-Melander (LU) </li>
</ul>
<p>Application round: Jan 2017 </p>
<p>Subject: Molecular Medicine &amp; Biomarkers</p>
<ul>
<li>Origin and heterogeneity of the most primitive nervous system  PI: Igor Adameyko (KI) </li>
</ul>
<p>Application round: Jun 2016 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1073/pnas.1920469117&gt;"> Klimovich <em>et al.</em> <strong>Prototypical pacemaker neurons interact with the resident microbiota.</strong> <em>Proc Natl Acad Sci USA</em> 117:17854-17863 (2020) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop&gt;"> <strong>Single-cell RNA-seq pipeline</strong> – a Snakemake workflow for quality control and basic analysis of single-cell RNA-seq data (under development) </a></p>
<ul>
<li>Personalized profiling of adaptive immune genes  PI: Gunilla Karlsson Hedestam (KI) </li>
</ul>
<p>Application round: May 2017 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.3389/fimmu.2019.00660&gt;"> Vázquez Bernat <em>et al.</em> <strong>High-quality library preparation for NGS-based immunoglobulin germline gene inference and repertoire expression analysis.</strong> <em>Front Immunol</em> 10:660 (2019) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1084/jem.20191155&gt;"> Phad <em>et al.</em> <strong>Extensive dissemination and intraclonal maturation of HIV Env vaccine-induced B cell responses.</strong> <em>J Exp Med</em> 217:e20191155 (2020) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;http://docs.igdiscover.se/&gt;"> <strong>IgDiscover</strong> – Software for antibody repertoire analysis and V gene discovery from high-throughput sequencing reads </a></p>
<ul>
<li>Plant spatial transcriptomics  PI: Nathaniel Street (UmU) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/nplants.2017.61&gt;"> Giacomello <em>et al.</em> <strong>Spatially resolved transcriptome profiling in model plant species.</strong> <em>Nat Plants</em> 3:17061 (2017) </a></p>
<ul>
<li>SCAPIS human wellness profiling  PI: Mathias Uhlén (KTH) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-020-18148-7&gt;"> Tebani <em>et al.</em> <strong>Integration of molecular profiles in a longitudinal wellness profiling cohort.</strong> <em>Nat Commun</em> 11:4487 (2020) </a></p>
<ul>
<li>Sequencing the missing link in land plant evolution  PI: Hans Ronne (SLU) </li>
</ul>
<p>Application round: Oct 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<ul>
<li>Single nuclei sequencing of arbuscular mycorrhizal fungi  PI: Anna Rosling (UU) </li>
</ul>
<p>Application round: May 2017 </p>
<p>Subject: Biodiversity</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41598-020-58025-3&gt;"> Montoliu-Nerin <em>et al.</em> <strong>Building de novo reference genome assemblies of complex eukaryotic microorganisms from single nuclei.</strong> <em>Sci Rep</em> 10:1303 (2020) </a></p>
<ul>
<li>Somatic genetic variation in the aging brain and Alzheimer's disease  PI: Maria Eriksson (KI) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1093/hmg/ddz085&gt;"> Helgadottir <em>et al.</em> <strong>Somatic mutation that affects transcription factor binding upstream of CD55 in the temporal cortex of a late-onset Alzheimer disease patient.</strong> <em>Hum Mol Genet</em> 28:2675–2685 (2019) </a></p>
<ul>
<li>Somatic mutations in tissue stem cells during human aging  PI: Maria Eriksson (KI) </li>
</ul>
<p>Application round: Jun 2015 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-018-03244-6&gt;"> Franco <em>et al.</em> <strong>Somatic mutagenesis in satellite cells associates with human skeletal muscle aging.</strong> <em>Nat Commun</em> 9:800 (2018) </a></p>
<ul>
<li>Spatial transcripts  PI: Jonas Frisén (KI) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: New Technology Development</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1126/science.aaf2403&gt;"> Ståhl <em>et al.</em> <strong>Visualization and analysis of gene expression in tissue sections by spatial transcriptomics.</strong> <em>Science</em> 353:78-82 (2016) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/ncomms13182&gt;"> Vickovic <em>et al.</em> <strong>Massive and parallel expression profiling using microarrayed single-cell sequencing.</strong> <em>Nat Commun</em> 7:13182 (2016) </a></p>
<ul>
<li>Spatially resolved gene expression analysis in human developing heart tissue  PI: Matthias Corbascio (KI) </li>
</ul>
<p>Application round: Oct 2016 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1016/j.cell.2019.11.025&gt;"> Asp <em>et al.</em> <strong>A spatiotemporal organ-wide gene expression and cell atlas of the developing human heart.</strong> <em>Cell</em> 179:1647-1660.e19 (2019) </a></p>
<ul>
<li>Speciation genomics in crows  PI: Jochen Wolf (UU) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1126/science.1253226&gt;"> Poelstra <em>et al.</em> <strong>The genomic landscape underlying phenotypic integrity in the face of gene flow in crows.</strong> <em>Science</em> 344:1410-1414 (2014) </a></p>
<ul>
<li>Stem cell mutational signatures  PI: Maria Eriksson (KI) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s13059-019-1892-z&gt;"> Franco <em>et al.</em> <strong>Whole genome DNA sequencing provides an atlas of somatic mutagenesis in healthy human cells and identifies a tumor-prone cell type.</strong> <em>Genome Biol</em> 20:285 (2019) </a></p>
<ul>
<li>The Swedish reference database of genetic variants  PI: Ulf Gyllensten (UU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Human Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/ejhg.2017.130&gt;"> Ameur <em>et al.</em> <strong>SweGen: a whole-genome data resource of genetic variability in a cross-section of the Swedish population.</strong> <em>Eur J Hum Genet</em> 25:1253-1260 (2017) </a> <a href="/support/ltsprojects/&lt;http://dx.doi.org/10.3390/genes9100486&gt;"> Ameur <em>et al.</em> <strong>De novo assembly of two Swedish genomes reveals missing segments from the human GRCh38 reference and improves variant calling of population-scale sequencing data.</strong> <em>Genes</em> 9:486 (2018) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/NBISweden/wgs-structvar&gt;"> <strong>Structural variation pipeline</strong> – A workflow for standardized calling and filtering of structural variants, designed for whole genome sequencing projects </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://swefreq.nbis.se&gt;">Genetic variant frequencies</a></p>
<ul>
<li>The herring genome  PI: Leif Andersson (UU) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.7554/eLife.12081&gt;"> Martinez Barrio <em>et al.</em> <strong>The genetic basis for ecological adaptation of the Atlantic herring revealed by genome sequencing.</strong> <em>Elife</em> 5:e12081 (2016) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;http://www.ncbi.nlm.nih.gov/assembly/318681&gt;">Genome assembly</a> <a href="/support/ltsprojects/&lt;https://doi.org/10.5061/dryad.5r774&gt;">Gene annotation and genetic variants</a> <a href="/support/ltsprojects/&lt;http://www.ncbi.nlm.nih.gov/sra/?term=SRP017094%3E">Population sequencing reads, set 1</a> <a href="/support/ltsprojects/&lt;http://www.ncbi.nlm.nih.gov/sra/?term=SRP017095%3E">Population sequencing reads, set 2</a> <a href="/support/ltsprojects/&lt;http://www.ncbi.nlm.nih.gov/sra/?term=SRP056617%3E">Population sequencing reads, set 3</a></p>
<ul>
<li>The transcriptional heterogeneity of human intestinal innate lymphoid cells  PI: Jenny Mjösberg (KI) </li>
</ul>
<p>Application round: Sep 2017 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41422-020-00445-x&gt;"> Mazzurana <em>et al.</em> <strong>Tissue-specific transcriptional imprinting and heterogeneity in human innate lymphoid cells revealed by full-length single-cell RNA-sequencing.</strong> <em>Cell Res</em> 31:554-568 (2021) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/czarnewski/ilc_tissue_imprinting&gt;"> <strong>Analysis scripts</strong> – to reproduce the analyses reported in the publication </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150050%3E">Gene expression data</a></p>
<ul>
<li>Transcriptomics of plant-insect coevolution  PI: Niklas Janz (SU) </li>
</ul>
<p>Application round: Oct 2013 </p>
<p>Subject: Evolutionary Biology/Genetics</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1186/s12862-016-0627-y&gt;"> Celorio-Mancera <em>et al.</em> <strong>Evolutionary history of host use, rather than plant phylogeny, determines gene expression in a generalist butterfly.</strong> <em>BMC Evol Biol</em> 16:59 (2016) </a></p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;https://github.com/hussius/butterfly&gt;"> <strong>Gene expression analysis</strong> – R and python scripts </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3861&gt;">RNA-seq data</a></p>
<ul>
<li>Travel and antibiotic resistance  PI: Joakim Larsson (GU) </li>
</ul>
<p>Application round: Jan 2013 </p>
<p>Subject: Biodiversity</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1128/AAC.00933-15&gt;"> Bengtsson-Palme <em>et al.</em> <strong>The human gut microbiome as a transporter of antibiotic resistance genes between continents.</strong> <em>Antimicrob Agents Chemother</em> 59:6551-6560 (2015) </a></p>
<ul>
<li>Tumorgan single cell sequencing  PI: Kristian Pietras (LU) </li>
</ul>
<p>Application round: Feb 2016 </p>
<p>Subject: Cancer Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/s41467-018-07582-3&gt;"> Bartoschek <em>et al.</em> <strong>Spatially and functionally distinct subclasses of breast cancer-associated fibroblasts revealed by single cell RNA sequencing.</strong> <em>Nat Commun</em> 9:5150 (2018) </a></p>
<ul>
<li>Whole genome sequencing of neuroblastoma  PI: Tommy Martinsson (GU) </li>
</ul>
<p>Application round: Oct 2015 </p>
<p>Subject: Cancer Biology</p>
<p>Tools and scripts:</p>
<p><a href="/support/ltsprojects/&lt;http://opensource.scilifelab.se/projects/sarek/&gt;"> <strong>Sarek</strong> – Complete open source pipeline to detect germline or somatic variants from normal or tumour/normal whole-genome sequencing data. </a></p>
<ul>
<li>Zebrafish mutation identification  PI: Olov Andersson (KI) </li>
</ul>
<p>Application round: Feb 2014 </p>
<p>Subject: Cell and Developmental Biology</p>
<p>Publications:</p>
<p><a href="/support/ltsprojects/&lt;http://dx.doi.org/10.1038/nature18614&gt;"> Reischauer <em>et al.</em> <strong>Cloche is a bHLH-PAS transcription factor that drives haemato-vascular specification.</strong> <em>Nature</em> 535:294-298 (2016) </a></p>
<p>Data: <a href="/support/ltsprojects/&lt;http://www.ncbi.nlm.nih.gov/sra/?term=PRJEB13442%3E">Whole-genome sequencing of zebrafish on PacBio</a> <a href="/support/ltsprojects/&lt;https://www.ncbi.nlm.nih.gov/nuccore/LT571435&gt;">Assembly of the Cloche/npas4l locus</a></p>

                </section>
            </section>
                </section>

    </div>

            <section id="footer" class="section bg-gray">
    <section class="container grid-lg">
        <p><a href="http://getgrav.org">Grav</a> was <i class="fa fa-code"></i> with <i class="fa fa-heart-o pulse "></i> by <a href="https://trilby.media">Trilby Media</a>.</p>
    </section>
</section>
    
        <div class="mobile-container">
        <div class="overlay" id="overlay">
            <div class="mobile-logo">
                <a href="/" class="navbar-brand mr-10">
  <svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" viewBox="0 0 504 140" clip-rule="evenodd"><path d="M235.83 71.56h-7.98c-1.2 0-2.2 1-2.2 2.2V89.1l-.15.13c-4.7 3.96-10.64 6.14-16.72 6.14-14.36 0-26.04-11.68-26.04-26.04s11.68-26.04 26.04-26.04c5.58 0 10.92 1.76 15.44 5.1.87.66 2.1.57 2.86-.2l5.7-5.7c.44-.44.67-1.05.63-1.68-.02-.62-.32-1.2-.82-1.6-6.76-5.35-15.2-8.3-23.8-8.3-21.18 0-38.42 17.23-38.42 38.4 0 21.2 17.24 38.42 38.42 38.42 10.93 0 21.4-4.7 28.7-12.9.35-.4.55-.93.55-1.47v-19.6c0-1.22-.98-2.2-2.2-2.2M502.8 34.44c-.4-.6-1.1-.98-1.84-.98h-8.7c-.87 0-1.66.52-2 1.32l-24.5 56.84-24.9-56.85c-.36-.8-1.15-1.3-2.02-1.3h-8.72c-.74 0-1.44.36-1.84.98-.4.62-.48 1.4-.17 2.1l30.2 68.85c.34.8 1.13 1.32 2 1.32h11c.88 0 1.67-.53 2.02-1.33l29.66-68.87c.3-.68.22-1.47-.2-2.1"/><path d="M388.68 34.77c-.35-.8-1.14-1.32-2-1.32h-11c-.88 0-1.67.53-2.02 1.33L344 103.64c-.3.68-.22 1.47.18 2.08.4.62 1.1 1 1.84 1h8.7c.86 0 1.66-.53 2-1.33l24.5-56.86 24.9 56.86c.36.8 1.15 1.32 2.02 1.32h8.72c.74 0 1.44-.38 1.84-1 .4-.62.47-1.4.17-2.1l-30.2-68.85zM309.2 81.52l.47-.22c8.68-4.2 14.28-13.1 14.28-22.67 0-13.88-11.3-25.18-25.17-25.18H266.9c-1.2 0-2.2 1-2.2 2.2v68.86c0 1.23 1 2.22 2.2 2.22h8c1.2 0 2.2-1 2.2-2.2V45.8h21.68c7.05 0 12.8 5.75 12.8 12.8 0 5.9-4 11-9.73 12.42-1.04.26-2.07.4-3.07.4h-7.98c-.83 0-1.6.46-1.96 1.2-.38.73-.3 1.62.2 2.3l22.6 30.87c.42.58 1.08.92 1.78.92h9.9c.84 0 1.6-.47 1.97-1.2.37-.75.3-1.64-.2-2.3l-15.9-21.7zM107.2 80.97c-7.26-4.8-11.4-8.85-15.02-16.1-2.47 4.97-8.24 12.37-17.96 18.2-4.86 15.1-27.96 44-35.43 39.9-2.22-1.2-2.64-2.8-2.15-4.45.54-4.13 9.08-13.62 9.08-13.62s.18 2 2.92 6.18c-3.6-11.2 5.96-25.03 8.5-29.73 3.98-1.27 4.27-6.4 4.27-6.4.26-7.9-3.28-13.63-6.7-17.05 2.46 3 3.25 7.54 3.37 11.7v.02c0 .47 0 .93 0 1.4-.12 3.43-1.16 8.18-3.38 8.18v.03c-2.28-.1-5.1.4-7.63 1.18l-5.6 1.34s2.98-.13 4.6 1.25c-1.8 2.9-5.78 6.53-10.22 8.58-6.45 3-8.3-2.96-5.03-6.84.8-.94 1.62-1.74 2.38-2.4-.5-.5-.8-1.2-.88-2.06 0 0 0 0 0-.02-.46-1.97-.2-4.54 2.6-8.62.54-.86 1.2-1.75 2-2.65.02-.04.04-.07.07-.1.03-.04.07-.08.1-.12.02-.02.04-.04.06-.06.2-.23.42-.45.64-.67 3.34-3.4 8.6-6.96 16.9-10.15C64.4 43.68 67.94 41 67.94 41c1.07-1.1 2.94-2.45 3.63-2.8-5.05-8.77-6.07-21.15-4.75-24.5-.1.2-.2.38-.3.57.5-1.14.83-1.5 1.34-2.1 1.38-1.64 6.06-2.5 7.74.96.9 1.84 1.06 4.23 1.03 6.02-3.7-.2-7.06 4.04-7.06 4.04s3.07-1.46 6.88-1.5c0 0 1 .9 2.28 2.56-1.7 3.2-4.52 10.02-2.5 17.16.35 1.4.86 2.62 1.5 3.65.02.05.04.1.07.14.05.07.1.13.14.2 3.37 5.06 9.54 5.66 9.54 5.66-2.9-1.45-5.27-3.76-6.8-6.56-.82-1.5-1.3-2.77-1.6-3.77-1.64-6.3.77-10 2.14-12.47 3.17-4.9 8.95-7.9 15.15-7.18 8.72 1 14.97 8.86 13.98 17.57-.6 5.32-3.78 9.72-8.15 12.12 1.05 2.84-.07 6.28-.07 6.28 2.64 3.32 2.76 5.23 2.67 7-3.36-.55-6.62 1.7-6.62 1.7s6.48-1.53 10.24 1.82c2.44 2.64 4.08 5 5.05 6.77 1.4 2.5 7.86 2.68 7.12 7.2-.74 4.5-5.68 4.53-13.4-.57M69.56 0C31.15 0 0 31.15 0 69.57c0 38.42 31.15 69.57 69.57 69.57 38.42 0 69.57-31.15 69.57-69.57C139.14 31.15 108 0 69.57 0M73.8 51.7c.8-.82.8-2.14 0-2.95-.82-.82-2.14-.82-2.95 0-.82.8-.82 2.13 0 2.94.8.8 2.13.8 2.95 0M66.45 53.15c-.82.8-.82 2.13 0 2.95.8.8 2.13.8 2.94 0 .8-.82.8-2.14 0-2.95-.82-.8-2.14-.8-2.95 0"/><path d="M79.23 54.23c-1.27-1.27-3.34-1.27-4.6 0l-2.72 2.7c-1.27 1.3-1.27 3.35 0 4.63l3 2.97c1.26 1.28 3.32 1.28 4.6 0l2.7-2.7c1.28-1.28 1.28-3.35 0-4.62l-2.97-2.97zM95.76 41.44c-2.15-2.57 1.87-7.25 4.4-4.46 4.64 5.15-2.25 7.04-4.4 4.46m9.24 2.7c3.45-6.56-1.42-10.4-4.77-13.53-5.36-5.03-10.7-7.2-16.8-.23-6.1 6.98-2.24 15.07 3.35 19.06 5.58 4 14.78 1.25 18.22-5.3"/></svg></a>
            </div>
            <nav class="overlay-menu">
                
<ul class="tree">
        
</ul>

            </nav>
        </div>
    </div>
    
    <script src="/assets/user/themes/quark/js/jquery.treemenu.js"></script>
<script src="/assets/user/themes/quark/js/site.js"></script>


</body>
</html>
