<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8" />
    <title>Juliana assis | NBIS - National Bioinformatics Infrastructure Sweden</title>

    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="generator" content="GravCMS" />
    <meta name="description" content="NBIS (National Bioinformatics Infrastructure Sweden) is a distributed national research infrastructure supported by the Swedish Research Council (Vetenskapsrådet), Science for Life Laboratory, all major Swedish universities and the Knut and Alice Wallenberg Foundation, providing state-of-the-art bioinformatics to the Swedish life science researchers community. NBIS is also the Swedish contact point to the European infrastructure for biological information ELIXIR." />

    <link rel="icon" type="image/png" href="/user/themes/quark/images/favicon.png" />
    <link rel="canonical" href="http://localhost/about/staff/juliana-assis" />

                        
            
    <link href="/assets/user/themes/quark/css-compiled/spectre.min.css" type="text/css" rel="stylesheet">
<link href="/assets/user/themes/quark/css-compiled/theme.min.css" type="text/css" rel="stylesheet">
<link href="/assets/user/themes/quark/css/custom.css" type="text/css" rel="stylesheet">
<link href="/assets/user/themes/quark/css/line-awesome.min.css" type="text/css" rel="stylesheet">

    <script src="/assets/system/assets/jquery/jquery-3.x.min.js"></script>

</head>
<body id="top" class=" header-fixed header-animated sticky-footer">
    <div id="page-wrapper">
            <section id="header" class="section">
            <section class="container grid-lg">
                <nav class="navbar">
                    <section class="navbar-section logo">
                        <a href="/" class="navbar-brand mr-10">
  <svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" viewBox="0 0 504 140" clip-rule="evenodd"><path d="M235.83 71.56h-7.98c-1.2 0-2.2 1-2.2 2.2V89.1l-.15.13c-4.7 3.96-10.64 6.14-16.72 6.14-14.36 0-26.04-11.68-26.04-26.04s11.68-26.04 26.04-26.04c5.58 0 10.92 1.76 15.44 5.1.87.66 2.1.57 2.86-.2l5.7-5.7c.44-.44.67-1.05.63-1.68-.02-.62-.32-1.2-.82-1.6-6.76-5.35-15.2-8.3-23.8-8.3-21.18 0-38.42 17.23-38.42 38.4 0 21.2 17.24 38.42 38.42 38.42 10.93 0 21.4-4.7 28.7-12.9.35-.4.55-.93.55-1.47v-19.6c0-1.22-.98-2.2-2.2-2.2M502.8 34.44c-.4-.6-1.1-.98-1.84-.98h-8.7c-.87 0-1.66.52-2 1.32l-24.5 56.84-24.9-56.85c-.36-.8-1.15-1.3-2.02-1.3h-8.72c-.74 0-1.44.36-1.84.98-.4.62-.48 1.4-.17 2.1l30.2 68.85c.34.8 1.13 1.32 2 1.32h11c.88 0 1.67-.53 2.02-1.33l29.66-68.87c.3-.68.22-1.47-.2-2.1"/><path d="M388.68 34.77c-.35-.8-1.14-1.32-2-1.32h-11c-.88 0-1.67.53-2.02 1.33L344 103.64c-.3.68-.22 1.47.18 2.08.4.62 1.1 1 1.84 1h8.7c.86 0 1.66-.53 2-1.33l24.5-56.86 24.9 56.86c.36.8 1.15 1.32 2.02 1.32h8.72c.74 0 1.44-.38 1.84-1 .4-.62.47-1.4.17-2.1l-30.2-68.85zM309.2 81.52l.47-.22c8.68-4.2 14.28-13.1 14.28-22.67 0-13.88-11.3-25.18-25.17-25.18H266.9c-1.2 0-2.2 1-2.2 2.2v68.86c0 1.23 1 2.22 2.2 2.22h8c1.2 0 2.2-1 2.2-2.2V45.8h21.68c7.05 0 12.8 5.75 12.8 12.8 0 5.9-4 11-9.73 12.42-1.04.26-2.07.4-3.07.4h-7.98c-.83 0-1.6.46-1.96 1.2-.38.73-.3 1.62.2 2.3l22.6 30.87c.42.58 1.08.92 1.78.92h9.9c.84 0 1.6-.47 1.97-1.2.37-.75.3-1.64-.2-2.3l-15.9-21.7zM107.2 80.97c-7.26-4.8-11.4-8.85-15.02-16.1-2.47 4.97-8.24 12.37-17.96 18.2-4.86 15.1-27.96 44-35.43 39.9-2.22-1.2-2.64-2.8-2.15-4.45.54-4.13 9.08-13.62 9.08-13.62s.18 2 2.92 6.18c-3.6-11.2 5.96-25.03 8.5-29.73 3.98-1.27 4.27-6.4 4.27-6.4.26-7.9-3.28-13.63-6.7-17.05 2.46 3 3.25 7.54 3.37 11.7v.02c0 .47 0 .93 0 1.4-.12 3.43-1.16 8.18-3.38 8.18v.03c-2.28-.1-5.1.4-7.63 1.18l-5.6 1.34s2.98-.13 4.6 1.25c-1.8 2.9-5.78 6.53-10.22 8.58-6.45 3-8.3-2.96-5.03-6.84.8-.94 1.62-1.74 2.38-2.4-.5-.5-.8-1.2-.88-2.06 0 0 0 0 0-.02-.46-1.97-.2-4.54 2.6-8.62.54-.86 1.2-1.75 2-2.65.02-.04.04-.07.07-.1.03-.04.07-.08.1-.12.02-.02.04-.04.06-.06.2-.23.42-.45.64-.67 3.34-3.4 8.6-6.96 16.9-10.15C64.4 43.68 67.94 41 67.94 41c1.07-1.1 2.94-2.45 3.63-2.8-5.05-8.77-6.07-21.15-4.75-24.5-.1.2-.2.38-.3.57.5-1.14.83-1.5 1.34-2.1 1.38-1.64 6.06-2.5 7.74.96.9 1.84 1.06 4.23 1.03 6.02-3.7-.2-7.06 4.04-7.06 4.04s3.07-1.46 6.88-1.5c0 0 1 .9 2.28 2.56-1.7 3.2-4.52 10.02-2.5 17.16.35 1.4.86 2.62 1.5 3.65.02.05.04.1.07.14.05.07.1.13.14.2 3.37 5.06 9.54 5.66 9.54 5.66-2.9-1.45-5.27-3.76-6.8-6.56-.82-1.5-1.3-2.77-1.6-3.77-1.64-6.3.77-10 2.14-12.47 3.17-4.9 8.95-7.9 15.15-7.18 8.72 1 14.97 8.86 13.98 17.57-.6 5.32-3.78 9.72-8.15 12.12 1.05 2.84-.07 6.28-.07 6.28 2.64 3.32 2.76 5.23 2.67 7-3.36-.55-6.62 1.7-6.62 1.7s6.48-1.53 10.24 1.82c2.44 2.64 4.08 5 5.05 6.77 1.4 2.5 7.86 2.68 7.12 7.2-.74 4.5-5.68 4.53-13.4-.57M69.56 0C31.15 0 0 31.15 0 69.57c0 38.42 31.15 69.57 69.57 69.57 38.42 0 69.57-31.15 69.57-69.57C139.14 31.15 108 0 69.57 0M73.8 51.7c.8-.82.8-2.14 0-2.95-.82-.82-2.14-.82-2.95 0-.82.8-.82 2.13 0 2.94.8.8 2.13.8 2.95 0M66.45 53.15c-.82.8-.82 2.13 0 2.95.8.8 2.13.8 2.94 0 .8-.82.8-2.14 0-2.95-.82-.8-2.14-.8-2.95 0"/><path d="M79.23 54.23c-1.27-1.27-3.34-1.27-4.6 0l-2.72 2.7c-1.27 1.3-1.27 3.35 0 4.63l3 2.97c1.26 1.28 3.32 1.28 4.6 0l2.7-2.7c1.28-1.28 1.28-3.35 0-4.62l-2.97-2.97zM95.76 41.44c-2.15-2.57 1.87-7.25 4.4-4.46 4.64 5.15-2.25 7.04-4.4 4.46m9.24 2.7c3.45-6.56-1.42-10.4-4.77-13.53-5.36-5.03-10.7-7.2-16.8-.23-6.1 6.98-2.24 15.07 3.35 19.06 5.58 4 14.78 1.25 18.22-5.3"/></svg></a>
                    </section>
                    <section class="navbar-section desktop-menu">

                        <nav class="dropmenu animated">
                                                    
<ul >
        
</ul>

                                                </nav>

                        
                    </section>
                </nav>
            </section>
        </section>
        <div class="mobile-menu">
            <div class="button_container" id="toggle">
                <span class="top"></span>
                <span class="middle"></span>
                <span class="bottom"></span>
            </div>
        </div>
    
    
        <section id="start">
                    <section id="body-wrapper" class="section">
                <section class="container grid-lg">
                                            
                                            <p><img src="/assets/img/staff/juliana-assis.png" alt="Juliana Assis" /></p>
<h3>Juliana Assis</h3>
<p><em>genomics, transcriptomics, system biology, metabolomics, data integration</em></p>
<table>
<thead>
<tr>
<th><em>email</em></th>
<th>juliana.assis@nbis.se</th>
</tr>
</thead>
<tbody>
<tr>
<td><img src="/assets/img/orcid_24x24_bw.png" alt="" /></td>
<td><a href="https://orcid.org/0000-0001-5995-8684">https://orcid.org/0000-0001-5995-8684</a></td>
</tr>
</tbody>
</table>
<p>I am a Biologist with a master and doctor degree in bioinformatics, with extensive experience in Whole Genome Sequencing (short and long reads). I have been working with bioinformatics since 2009, starting in my undergraduate studies. My field lies broadly in the multi-omics data including Genomic, Transcriptomic, Metagenomic and currently data Integration.</p>
<p>I got my masters and Ph.D. in Bioinformatics at the Universidade Federal de Minas Gerais (UFMG, Brazil), with a collaborative period in the University of Georgia (UGA, USA), and at the Wellcome Sanger Institute, Sanger - UK.</p>
<p>The group where I developed my master’s and Ph.D. is a Platform of Bioinformatic which is a government agency under the Ministry of Health in Brazil(Oswaldo Cruz Foundation). The platform is responsible to help others researches to design the experiments and perform the bioinformatics analysis, as also to provide the infrastructure for the analysis. At this platform I was working with NGS (short and long reads) data processing,including genomic, RNA-seq, Hi-C, WGS, metagenome, variant calling. The group where I worked for 2 years in the USA, is responsible to develop the VEupathDB “Database Family”, which gave me the opportunity to learn the database infrastructure and the data integration. I also did a short training (a month) in the Wellcome Sanger Institute (Sanger,UK) to learn how to improve the quality of eukaryotic genome annotation.</p>
<p>In 2019, I moved to Copenhagen - Denmark. During my Postdoctoral Researcher at the Novo Nordisk Foundation Center for Basic Metabolic Research - CBMR, I worked in the Human microbiome field, with long reads - Nanopore, PacBio ISO-seq data and short reads (illumina) to perform metagenome assembly (binning methods) and integrating with the other omics-data (metatranscriptomics and metametabolomics). In addition, I was also involved in some 16S rRNA taxonomic studies. During the last couple of years, I have published papers involving the high quality of genomics studies.</p>
<h3>Selected publications</h3>
<p>Andrade, L. F. de, Mourao, M. de M., Geraldo, J. A., Coelho, F. S., Silva, L. L., Neves, R. H., Volpini, A., Machado-Silva, J. R., Araujo, N., Nacif-Pimenta, R., and others (2014) Regulation of Schistosoma mansoni development and reproduction by the mitogen-activated protein kinase signaling pathway. <em>PLoS neglected tropical diseases</em> , <strong>8</strong> , e2949.  </p>
<p>Pylro, V. S., Oliveira, F. S., Morais, D. K., Cuadros-Orellana, S., Pais, F. S.-M., Medeiros, J. D., Geraldo, J. A., Gilbert, J., Volpini, A. C., and Fernandes, G. R. (2016) ZIKV–CDB: a collaborative database to guide research linking SncRNAs and ZIKA virus disease symptoms. <em>PLoS neglected tropical diseases</em> , <strong>10</strong> , e0004817.  </p>
<p>Chien, J.-T., Pakala, S. B., Geraldo, J. A., Lapp, S. A., Humphrey, J. C., Barnwell, J. W., Kissinger, J. C., and Galinski, M. R. (2016) High-quality genome assembly and annotation for Plasmodium coatneyi, generated using single-molecule real-time PacBio technology. <em>Genome announcements</em> , <strong>4</strong> , e00883–16.  </p>
<p>Adema, C. M., Hillier, L. D. W., Jones, C. S., Loker, E. S., Knight, M., Minx, P., Oliveira, G., Raghavan, N., Shedlock, A., Do Amaral, L. R., and others (2017) Whole genome analysis of a schistosomiasis-transmitting freshwater snail. <em>Nature communications</em> , <strong>8</strong> , 1–12.  </p>
<p>Maldonado, L. L., Assis, J., Araújo, F. M. G., Salim, A. C. M., Macchiaroli, N., Cucher, M., Camicia, F., Fox, A., Rosenzvit, M., Oliveira, G., and others (2017) The Echinococcus canadensis (G7) genome: a key knowledge of parasitic platyhelminth human diseases. <em>Bmc Genomics</em> , <strong>18</strong> , 1–23.  </p>
<p>Rosse, I. C., Assis, J. G., Oliveira, F. S., Leite, L. R., Araujo, F., Zerlotini, A., Volpini, A., Dominitini, A. J., Lopes, B. C., Arbex, W. A., and others (2017) Whole genome sequencing of Guzerá cattle reveals genetic variants in candidate genes for production, disease resistance, and heat tolerance. <em>Mammalian genome</em> , <strong>28</strong> , 66–80.  </p>
<p>Lapp, S. A., Geraldo, J. A., Chien, J.-T., Ay, F., Pakala, S. B., Batugedara, G., Humphrey, J., DeBARRY, J. D., Le Roch, K. G., Galinski, M. R., and others (2018) PacBio assembly of a Plasmodium knowlesi genome sequence with Hi-C correction and manual annotation of the SICAvar gene family. <em>Parasitology</em> , <strong>145</strong> , 71–84.  </p>
<p>Figueiró, H. V., Li, G., Trindade, F. J., Assis, J., Pais, F., Fernandes, G., Santos, S. H. D., Hughes, G. M., Komissarov, A., Antunes, A., and others (2017) Genome-wide signatures of complex introgression and adaptive evolution in the big cats. <em>Science Advances</em> , <strong>3</strong> , e1700299.  </p>
<p>Geraldo, J. A., Silva, F., Horácio, E. C. A., Leão, A., Brayan, A., Rocha Fernandes, G. da, and Coimbra, N. D. R. (2018) Highlights of the 2nd Brazilian student council symposium, organized by ISCB Regional Student Group Brazil. <em>PeerJ PrePrints</em>.  </p>
<p>Brito Raı́ssa N, Geraldo, J. A., Monteiro, F. A., Lazoski, C., Souza, R. C. M., and Abad-Franch, F. (2019) Transcriptome-based molecular systematics: Rhodnius montenegrensis (Triatominae) and its position within the Rhodnius prolixus–Rhodnius robustus cryptic–species complex. <em>Parasites &amp; vectors</em>, <strong>12</strong> , 1–16.  </p>
<p>Martinez-Villegas, L., Assis-Geraldo, J., Koerich, L. B., Collier, T. C., Lee, Y., Main, B. J., Rodrigues, N. B., Orfano, A. S., Pires, A. C. A. M., Campolina, T. B., and others (2019) Characterization of the complete mitogenome of Anopheles aquasalis, and phylogenetic divergences among Anopheles from diverse geographic zones. <em>PloS one</em> , <strong>14</strong> , e0219523.  </p>
<p>Shome, S., Parra, R. G., Fatima, N., Monzon, A. M., Cuypers, B., Moosa, Y., Coimbra, N. D. R., Assis, J., Giner-Delgado, C., Dönertaş, H. M., and others (2019) Global network of computational biology communities: ISCB’s Regional Student Groups breaking barriers. <em>F1000Research</em> , <strong>8</strong>.  </p>
<p>Airey, E., Portelli, S., Xavier, J. S., Myung, Y. C., Silk, M., Karmakar, M., Velloso, J. P. L., Rodrigues, C. H. M., Parate, H. H., Garg, A., and others (2021) Identifying Genotype–Phenotype Correlations via Integrative Mutation Analysis. In, <em>Artificial Neural Networks</em>. Humana, New York, NY, pp. 1–32.  </p>
<p>Gnanasekaran, T., Assis Geraldo, J., Ahrenkiel, D. W., Alvarez-Silva, C., Saenz, C., Khan, A., Hanteer, O., Gunalan, V., Trost, K., Moritz, T., and others (2021) Ecological Adaptation and Succession of Human Fecal Microbial Communities in an Automated In Vitro Fermentation System. <em>Msystems</em> , <strong>6</strong> , e00232–21. </p>
<p><img src="/assets/img/staff/juliana-assis.png" alt="Juliana Assis" /></p>
<h3>Juliana Assis</h3>
<p><em>genomics, transcriptomics, system biology, metabolomics, data integration</em></p>
<table>
<thead>
<tr>
<th><em>email</em></th>
<th>juliana.assis@nbis.se</th>
</tr>
</thead>
<tbody>
<tr>
<td><img src="/assets/img/orcid_24x24_bw.png" alt="" /></td>
<td><a href="https://orcid.org/0000-0001-5995-8684">https://orcid.org/0000-0001-5995-8684</a></td>
</tr>
</tbody>
</table>
<p>I am a Biologist with a master and doctor degree in bioinformatics, with extensive experience in Whole Genome Sequencing (short and long reads). I have been working with bioinformatics since 2009, starting in my undergraduate studies. My field lies broadly in the multi-omics data including Genomic, Transcriptomic, Metagenomic and currently data Integration.</p>
<p>I got my masters and Ph.D. in Bioinformatics at the Universidade Federal de Minas Gerais (UFMG, Brazil), with a collaborative period in the University of Georgia (UGA, USA), and at the Wellcome Sanger Institute, Sanger - UK.</p>
<p>The group where I developed my master’s and Ph.D. is a Platform of Bioinformatic which is a government agency under the Ministry of Health in Brazil(Oswaldo Cruz Foundation). The platform is responsible to help others researches to design the experiments and perform the bioinformatics analysis, as also to provide the infrastructure for the analysis. At this platform I was working with NGS (short and long reads) data processing,including genomic, RNA-seq, Hi-C, WGS, metagenome, variant calling. The group where I worked for 2 years in the USA, is responsible to develop the VEupathDB “Database Family”, which gave me the opportunity to learn the database infrastructure and the data integration. I also did a short training (a month) in the Wellcome Sanger Institute (Sanger,UK) to learn how to improve the quality of eukaryotic genome annotation.</p>
<p>In 2019, I moved to Copenhagen - Denmark. During my Postdoctoral Researcher at the Novo Nordisk Foundation Center for Basic Metabolic Research - CBMR, I worked in the Human microbiome field, with long reads - Nanopore, PacBio ISO-seq data and short reads (illumina) to perform metagenome assembly (binning methods) and integrating with the other omics-data (metatranscriptomics and metametabolomics). In addition, I was also involved in some 16S rRNA taxonomic studies. During the last couple of years, I have published papers involving the high quality of genomics studies.</p>
<h3>Selected publications</h3>
<p>Andrade, L. F. de, Mourao, M. de M., Geraldo, J. A., Coelho, F. S., Silva, L. L., Neves, R. H., Volpini, A., Machado-Silva, J. R., Araujo, N., Nacif-Pimenta, R., and others (2014) Regulation of Schistosoma mansoni development and reproduction by the mitogen-activated protein kinase signaling pathway. <em>PLoS neglected tropical diseases</em> , <strong>8</strong> , e2949.  </p>
<p>Pylro, V. S., Oliveira, F. S., Morais, D. K., Cuadros-Orellana, S., Pais, F. S.-M., Medeiros, J. D., Geraldo, J. A., Gilbert, J., Volpini, A. C., and Fernandes, G. R. (2016) ZIKV–CDB: a collaborative database to guide research linking SncRNAs and ZIKA virus disease symptoms. <em>PLoS neglected tropical diseases</em> , <strong>10</strong> , e0004817.  </p>
<p>Chien, J.-T., Pakala, S. B., Geraldo, J. A., Lapp, S. A., Humphrey, J. C., Barnwell, J. W., Kissinger, J. C., and Galinski, M. R. (2016) High-quality genome assembly and annotation for Plasmodium coatneyi, generated using single-molecule real-time PacBio technology. <em>Genome announcements</em> , <strong>4</strong> , e00883–16.  </p>
<p>Adema, C. M., Hillier, L. D. W., Jones, C. S., Loker, E. S., Knight, M., Minx, P., Oliveira, G., Raghavan, N., Shedlock, A., Do Amaral, L. R., and others (2017) Whole genome analysis of a schistosomiasis-transmitting freshwater snail. <em>Nature communications</em> , <strong>8</strong> , 1–12.  </p>
<p>Maldonado, L. L., Assis, J., Araújo, F. M. G., Salim, A. C. M., Macchiaroli, N., Cucher, M., Camicia, F., Fox, A., Rosenzvit, M., Oliveira, G., and others (2017) The Echinococcus canadensis (G7) genome: a key knowledge of parasitic platyhelminth human diseases. <em>Bmc Genomics</em> , <strong>18</strong> , 1–23.  </p>
<p>Rosse, I. C., Assis, J. G., Oliveira, F. S., Leite, L. R., Araujo, F., Zerlotini, A., Volpini, A., Dominitini, A. J., Lopes, B. C., Arbex, W. A., and others (2017) Whole genome sequencing of Guzerá cattle reveals genetic variants in candidate genes for production, disease resistance, and heat tolerance. <em>Mammalian genome</em> , <strong>28</strong> , 66–80.  </p>
<p>Lapp, S. A., Geraldo, J. A., Chien, J.-T., Ay, F., Pakala, S. B., Batugedara, G., Humphrey, J., DeBARRY, J. D., Le Roch, K. G., Galinski, M. R., and others (2018) PacBio assembly of a Plasmodium knowlesi genome sequence with Hi-C correction and manual annotation of the SICAvar gene family. <em>Parasitology</em> , <strong>145</strong> , 71–84.  </p>
<p>Figueiró, H. V., Li, G., Trindade, F. J., Assis, J., Pais, F., Fernandes, G., Santos, S. H. D., Hughes, G. M., Komissarov, A., Antunes, A., and others (2017) Genome-wide signatures of complex introgression and adaptive evolution in the big cats. <em>Science Advances</em> , <strong>3</strong> , e1700299.  </p>
<p>Geraldo, J. A., Silva, F., Horácio, E. C. A., Leão, A., Brayan, A., Rocha Fernandes, G. da, and Coimbra, N. D. R. (2018) Highlights of the 2nd Brazilian student council symposium, organized by ISCB Regional Student Group Brazil. <em>PeerJ PrePrints</em>.  </p>
<p>Brito Raı́ssa N, Geraldo, J. A., Monteiro, F. A., Lazoski, C., Souza, R. C. M., and Abad-Franch, F. (2019) Transcriptome-based molecular systematics: Rhodnius montenegrensis (Triatominae) and its position within the Rhodnius prolixus–Rhodnius robustus cryptic–species complex. <em>Parasites &amp; vectors</em>, <strong>12</strong> , 1–16.  </p>
<p>Martinez-Villegas, L., Assis-Geraldo, J., Koerich, L. B., Collier, T. C., Lee, Y., Main, B. J., Rodrigues, N. B., Orfano, A. S., Pires, A. C. A. M., Campolina, T. B., and others (2019) Characterization of the complete mitogenome of Anopheles aquasalis, and phylogenetic divergences among Anopheles from diverse geographic zones. <em>PloS one</em> , <strong>14</strong> , e0219523.  </p>
<p>Shome, S., Parra, R. G., Fatima, N., Monzon, A. M., Cuypers, B., Moosa, Y., Coimbra, N. D. R., Assis, J., Giner-Delgado, C., Dönertaş, H. M., and others (2019) Global network of computational biology communities: ISCB’s Regional Student Groups breaking barriers. <em>F1000Research</em> , <strong>8</strong>.  </p>
<p>Airey, E., Portelli, S., Xavier, J. S., Myung, Y. C., Silk, M., Karmakar, M., Velloso, J. P. L., Rodrigues, C. H. M., Parate, H. H., Garg, A., and others (2021) Identifying Genotype–Phenotype Correlations via Integrative Mutation Analysis. In, <em>Artificial Neural Networks</em>. Humana, New York, NY, pp. 1–32.  </p>
<p>Gnanasekaran, T., Assis Geraldo, J., Ahrenkiel, D. W., Alvarez-Silva, C., Saenz, C., Khan, A., Hanteer, O., Gunalan, V., Trost, K., Moritz, T., and others (2021) Ecological Adaptation and Succession of Human Fecal Microbial Communities in an Automated In Vitro Fermentation System. <em>Msystems</em> , <strong>6</strong> , e00232–21. </p>

                </section>
            </section>
                </section>

    </div>

            <section id="footer" class="section bg-gray">
    <section class="container grid-lg">
        <p><a href="http://getgrav.org">Grav</a> was <i class="fa fa-code"></i> with <i class="fa fa-heart-o pulse "></i> by <a href="https://trilby.media">Trilby Media</a>.</p>
    </section>
</section>
    
        <div class="mobile-container">
        <div class="overlay" id="overlay">
            <div class="mobile-logo">
                <a href="/" class="navbar-brand mr-10">
  <svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" viewBox="0 0 504 140" clip-rule="evenodd"><path d="M235.83 71.56h-7.98c-1.2 0-2.2 1-2.2 2.2V89.1l-.15.13c-4.7 3.96-10.64 6.14-16.72 6.14-14.36 0-26.04-11.68-26.04-26.04s11.68-26.04 26.04-26.04c5.58 0 10.92 1.76 15.44 5.1.87.66 2.1.57 2.86-.2l5.7-5.7c.44-.44.67-1.05.63-1.68-.02-.62-.32-1.2-.82-1.6-6.76-5.35-15.2-8.3-23.8-8.3-21.18 0-38.42 17.23-38.42 38.4 0 21.2 17.24 38.42 38.42 38.42 10.93 0 21.4-4.7 28.7-12.9.35-.4.55-.93.55-1.47v-19.6c0-1.22-.98-2.2-2.2-2.2M502.8 34.44c-.4-.6-1.1-.98-1.84-.98h-8.7c-.87 0-1.66.52-2 1.32l-24.5 56.84-24.9-56.85c-.36-.8-1.15-1.3-2.02-1.3h-8.72c-.74 0-1.44.36-1.84.98-.4.62-.48 1.4-.17 2.1l30.2 68.85c.34.8 1.13 1.32 2 1.32h11c.88 0 1.67-.53 2.02-1.33l29.66-68.87c.3-.68.22-1.47-.2-2.1"/><path d="M388.68 34.77c-.35-.8-1.14-1.32-2-1.32h-11c-.88 0-1.67.53-2.02 1.33L344 103.64c-.3.68-.22 1.47.18 2.08.4.62 1.1 1 1.84 1h8.7c.86 0 1.66-.53 2-1.33l24.5-56.86 24.9 56.86c.36.8 1.15 1.32 2.02 1.32h8.72c.74 0 1.44-.38 1.84-1 .4-.62.47-1.4.17-2.1l-30.2-68.85zM309.2 81.52l.47-.22c8.68-4.2 14.28-13.1 14.28-22.67 0-13.88-11.3-25.18-25.17-25.18H266.9c-1.2 0-2.2 1-2.2 2.2v68.86c0 1.23 1 2.22 2.2 2.22h8c1.2 0 2.2-1 2.2-2.2V45.8h21.68c7.05 0 12.8 5.75 12.8 12.8 0 5.9-4 11-9.73 12.42-1.04.26-2.07.4-3.07.4h-7.98c-.83 0-1.6.46-1.96 1.2-.38.73-.3 1.62.2 2.3l22.6 30.87c.42.58 1.08.92 1.78.92h9.9c.84 0 1.6-.47 1.97-1.2.37-.75.3-1.64-.2-2.3l-15.9-21.7zM107.2 80.97c-7.26-4.8-11.4-8.85-15.02-16.1-2.47 4.97-8.24 12.37-17.96 18.2-4.86 15.1-27.96 44-35.43 39.9-2.22-1.2-2.64-2.8-2.15-4.45.54-4.13 9.08-13.62 9.08-13.62s.18 2 2.92 6.18c-3.6-11.2 5.96-25.03 8.5-29.73 3.98-1.27 4.27-6.4 4.27-6.4.26-7.9-3.28-13.63-6.7-17.05 2.46 3 3.25 7.54 3.37 11.7v.02c0 .47 0 .93 0 1.4-.12 3.43-1.16 8.18-3.38 8.18v.03c-2.28-.1-5.1.4-7.63 1.18l-5.6 1.34s2.98-.13 4.6 1.25c-1.8 2.9-5.78 6.53-10.22 8.58-6.45 3-8.3-2.96-5.03-6.84.8-.94 1.62-1.74 2.38-2.4-.5-.5-.8-1.2-.88-2.06 0 0 0 0 0-.02-.46-1.97-.2-4.54 2.6-8.62.54-.86 1.2-1.75 2-2.65.02-.04.04-.07.07-.1.03-.04.07-.08.1-.12.02-.02.04-.04.06-.06.2-.23.42-.45.64-.67 3.34-3.4 8.6-6.96 16.9-10.15C64.4 43.68 67.94 41 67.94 41c1.07-1.1 2.94-2.45 3.63-2.8-5.05-8.77-6.07-21.15-4.75-24.5-.1.2-.2.38-.3.57.5-1.14.83-1.5 1.34-2.1 1.38-1.64 6.06-2.5 7.74.96.9 1.84 1.06 4.23 1.03 6.02-3.7-.2-7.06 4.04-7.06 4.04s3.07-1.46 6.88-1.5c0 0 1 .9 2.28 2.56-1.7 3.2-4.52 10.02-2.5 17.16.35 1.4.86 2.62 1.5 3.65.02.05.04.1.07.14.05.07.1.13.14.2 3.37 5.06 9.54 5.66 9.54 5.66-2.9-1.45-5.27-3.76-6.8-6.56-.82-1.5-1.3-2.77-1.6-3.77-1.64-6.3.77-10 2.14-12.47 3.17-4.9 8.95-7.9 15.15-7.18 8.72 1 14.97 8.86 13.98 17.57-.6 5.32-3.78 9.72-8.15 12.12 1.05 2.84-.07 6.28-.07 6.28 2.64 3.32 2.76 5.23 2.67 7-3.36-.55-6.62 1.7-6.62 1.7s6.48-1.53 10.24 1.82c2.44 2.64 4.08 5 5.05 6.77 1.4 2.5 7.86 2.68 7.12 7.2-.74 4.5-5.68 4.53-13.4-.57M69.56 0C31.15 0 0 31.15 0 69.57c0 38.42 31.15 69.57 69.57 69.57 38.42 0 69.57-31.15 69.57-69.57C139.14 31.15 108 0 69.57 0M73.8 51.7c.8-.82.8-2.14 0-2.95-.82-.82-2.14-.82-2.95 0-.82.8-.82 2.13 0 2.94.8.8 2.13.8 2.95 0M66.45 53.15c-.82.8-.82 2.13 0 2.95.8.8 2.13.8 2.94 0 .8-.82.8-2.14 0-2.95-.82-.8-2.14-.8-2.95 0"/><path d="M79.23 54.23c-1.27-1.27-3.34-1.27-4.6 0l-2.72 2.7c-1.27 1.3-1.27 3.35 0 4.63l3 2.97c1.26 1.28 3.32 1.28 4.6 0l2.7-2.7c1.28-1.28 1.28-3.35 0-4.62l-2.97-2.97zM95.76 41.44c-2.15-2.57 1.87-7.25 4.4-4.46 4.64 5.15-2.25 7.04-4.4 4.46m9.24 2.7c3.45-6.56-1.42-10.4-4.77-13.53-5.36-5.03-10.7-7.2-16.8-.23-6.1 6.98-2.24 15.07 3.35 19.06 5.58 4 14.78 1.25 18.22-5.3"/></svg></a>
            </div>
            <nav class="overlay-menu">
                
<ul class="tree">
        
</ul>

            </nav>
        </div>
    </div>
    
    <script src="/assets/user/themes/quark/js/jquery.treemenu.js"></script>
<script src="/assets/user/themes/quark/js/site.js"></script>


</body>
</html>
