<!doctype html><html lang=en><meta charset=utf-8><title>Tools | NBIS - National Bioinformatics Infrastructure Sweden</title><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><meta name=generator content="GravCMS"><meta name=description content="NBIS (National Bioinformatics Infrastructure Sweden) is a distributed national research infrastructure supported by the Swedish Research Council (Vetenskapsrådet), Science for Life Laboratory, all major Swedish universities and the Knut and Alice Wallenberg Foundation, providing state-of-the-art bioinformatics to the Swedish life science researchers community. NBIS is also the Swedish contact point to the European infrastructure for biological information ELIXIR."><link rel=icon type=image/png href=/user/themes/quark/images/favicon.png><link rel=canonical href=nbis-static.dahlo.se/infrastructure/tools><link href=/user/plugins/markdown-notices/assets/notices.css rel=stylesheet><link href=/user/plugins/form/assets/form-styles.css rel=stylesheet><link href=/user/plugins/login/css/login.css rel=stylesheet><link href=/user/themes/quark/css-compiled/spectre.min.css rel=stylesheet><link href=/user/themes/quark/css-compiled/theme.min.css rel=stylesheet><link href=/user/themes/quark/css/custom.css rel=stylesheet><link href=/user/themes/quark/css/line-awesome.min.css rel=stylesheet><script src=/system/assets/jquery/jquery-3.x.min.js></script><body id=top class="header-fixed header-animated sticky-footer"><div id=page-wrapper><section id=header class=section><section class="container grid-lg"><nav class=navbar><section class="navbar-section logo"><a href=/ class="navbar-brand mr-10"><svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" viewBox="0 0 504 140" clip-rule="evenodd"><path d="M235.83 71.56h-7.98c-1.2.0-2.2 1-2.2 2.2V89.1l-.15.13c-4.7 3.96-10.64 6.14-16.72 6.14-14.36.0-26.04-11.68-26.04-26.04s11.68-26.04 26.04-26.04c5.58.0 10.92 1.76 15.44 5.1.87.66 2.1.57 2.86-.2l5.7-5.7c.44-.44.67-1.05.63-1.68-.02-.62-.32-1.2-.82-1.6-6.76-5.35-15.2-8.3-23.8-8.3-21.18.0-38.42 17.23-38.42 38.4.0 21.2 17.24 38.42 38.42 38.42 10.93.0 21.4-4.7 28.7-12.9.35-.4.55-.93.55-1.47v-19.6c0-1.22-.98-2.2-2.2-2.2M502.8 34.44c-.4-.6-1.1-.98-1.84-.98h-8.7c-.87.0-1.66.52-2 1.32l-24.5 56.84-24.9-56.85c-.36-.8-1.15-1.3-2.02-1.3h-8.72c-.74.0-1.44.36-1.84.98s-.48 1.4-.17 2.1l30.2 68.85c.34.8 1.13 1.32 2 1.32h11c.88.0 1.67-.53 2.02-1.33l29.66-68.87c.3-.68.22-1.47-.2-2.1"/><path d="M388.68 34.77c-.35-.8-1.14-1.32-2-1.32h-11c-.88.0-1.67.53-2.02 1.33L344 103.64c-.3.68-.22 1.47.18 2.08.4.62 1.1 1 1.84 1h8.7c.86.0 1.66-.53 2-1.33l24.5-56.86 24.9 56.86c.36.8 1.15 1.32 2.02 1.32h8.72c.74.0 1.44-.38 1.84-1s.47-1.4.17-2.1l-30.2-68.85zM309.2 81.52l.47-.22c8.68-4.2 14.28-13.1 14.28-22.67.0-13.88-11.3-25.18-25.17-25.18H266.9c-1.2.0-2.2 1-2.2 2.2v68.86c0 1.23 1 2.22 2.2 2.22h8c1.2.0 2.2-1 2.2-2.2V45.8h21.68c7.05.0 12.8 5.75 12.8 12.8.0 5.9-4 11-9.73 12.42-1.04.26-2.07.4-3.07.4h-7.98c-.83.0-1.6.46-1.96 1.2-.38.73-.3 1.62.2 2.3l22.6 30.87c.42.58 1.08.92 1.78.92h9.9c.84.0 1.6-.47 1.97-1.2.37-.75.3-1.64-.2-2.3l-15.9-21.7zM107.2 80.97c-7.26-4.8-11.4-8.85-15.02-16.1-2.47 4.97-8.24 12.37-17.96 18.2-4.86 15.1-27.96 44-35.43 39.9-2.22-1.2-2.64-2.8-2.15-4.45.54-4.13 9.08-13.62 9.08-13.62s.18 2 2.92 6.18c-3.6-11.2 5.96-25.03 8.5-29.73 3.98-1.27 4.27-6.4 4.27-6.4.26-7.9-3.28-13.63-6.7-17.05 2.46 3 3.25 7.54 3.37 11.7v.02c0 .47.0.93.0 1.4-.12 3.43-1.16 8.18-3.38 8.18v.03c-2.28-.1-5.1.4-7.63 1.18l-5.6 1.34s2.98-.13 4.6 1.25c-1.8 2.9-5.78 6.53-10.22 8.58-6.45 3-8.3-2.96-5.03-6.84.8-.94 1.62-1.74 2.38-2.4-.5-.5-.8-1.2-.88-2.06v-.02c-.46-1.97-.2-4.54 2.6-8.62.54-.86 1.2-1.75 2-2.65.02-.04.04-.07.07-.1.03-.04.07-.08.1-.12.02-.02.04-.04.06-.06.2-.23.42-.45.64-.67 3.34-3.4 8.6-6.96 16.9-10.15C64.4 43.68 67.94 41 67.94 41c1.07-1.1 2.94-2.45 3.63-2.8-5.05-8.77-6.07-21.15-4.75-24.5-.1.2-.2.38-.3.57.5-1.14.83-1.5 1.34-2.1 1.38-1.64 6.06-2.5 7.74.96.9 1.84 1.06 4.23 1.03 6.02-3.7-.2-7.06 4.04-7.06 4.04s3.07-1.46 6.88-1.5c0 0 1 .9 2.28 2.56-1.7 3.2-4.52 10.02-2.5 17.16.35 1.4.86 2.62 1.5 3.65.02.05.04.1.07.14.05.07.1.13.14.2 3.37 5.06 9.54 5.66 9.54 5.66-2.9-1.45-5.27-3.76-6.8-6.56-.82-1.5-1.3-2.77-1.6-3.77-1.64-6.3.77-10 2.14-12.47 3.17-4.9 8.95-7.9 15.15-7.18 8.72 1 14.97 8.86 13.98 17.57-.6 5.32-3.78 9.72-8.15 12.12 1.05 2.84-.07 6.28-.07 6.28 2.64 3.32 2.76 5.23 2.67 7-3.36-.55-6.62 1.7-6.62 1.7s6.48-1.53 10.24 1.82c2.44 2.64 4.08 5 5.05 6.77 1.4 2.5 7.86 2.68 7.12 7.2-.74 4.5-5.68 4.53-13.4-.57M69.56.0C31.15.0.0 31.15.0 69.57c0 38.42 31.15 69.57 69.57 69.57 38.42.0 69.57-31.15 69.57-69.57C139.14 31.15 108 0 69.57.0M73.8 51.7c.8-.82.8-2.14.0-2.95-.82-.82-2.14-.82-2.95.0-.82.8-.82 2.13.0 2.94.8.8 2.13.8 2.95.0M66.45 53.15c-.82.8-.82 2.13.0 2.95.8.8 2.13.8 2.94.0.8-.82.8-2.14.0-2.95-.82-.8-2.14-.8-2.95.0"/><path d="M79.23 54.23c-1.27-1.27-3.34-1.27-4.6.0l-2.72 2.7c-1.27 1.3-1.27 3.35.0 4.63l3 2.97c1.26 1.28 3.32 1.28 4.6.0l2.7-2.7c1.28-1.28 1.28-3.35.0-4.62l-2.97-2.97zM95.76 41.44c-2.15-2.57 1.87-7.25 4.4-4.46 4.64 5.15-2.25 7.04-4.4 4.46m9.24 2.7c3.45-6.56-1.42-10.4-4.77-13.53-5.36-5.03-10.7-7.2-16.8-.23-6.1 6.98-2.24 15.07 3.35 19.06 5.58 4 14.78 1.25 18.22-5.3"/></svg></a></section><section class="navbar-section desktop-menu"><nav class="dropmenu animated"><ul><li><a href=/>Home</a><li><a href=/infrastructure class=active>Nbis infrastructure</a><ul><li><a href=/infrastructure/aida-data-hub>Aida data hub</a><li><a href=/infrastructure/compute-and-storage>Nbis compute and storage</a><li><a href=/infrastructure/data-management>Data management</a><ul><li><a href=/infrastructure/data-management/data-management.html>Data management</a></ul><li><a href=/infrastructure/general-information>General information</a><li><a href=/infrastructure/mosler>Mosler</a><li><a href=/infrastructure/sensitive-data-archive>Swedish sensitive data archive</a><li><a href=/infrastructure/tools class=active>Tools</a></ul><li><a href=/events>Events</a><ul><li><a href=/events/past-events>Previous nbis events</a><li><a href=/events/symposium2018>Nbis symposium and user meeting 2018</a></ul><li><a href=/search>Search</a><li><a href=/about>About nbis</a><ul><li><a href=/about/contact>Contact information</a><li><a href=/about/doc>Nbis documents</a><li><a href=/about/organization>Nbis organization</a><li><a href=/about/privacy>Privacy policy</a><li><a href=/about/staff>Nbis staff</a><li><a href=/about/status>Current status</a><li><a href=/about/vacancies>Vacancies</a></ul><li><a href=/blog>Blog</a><ul><li><a href=/blog/2018-02-22-cheating-for-speed.html>[when to cheat on your principles for speed, the dark art of getting things done](&lt;> "permalink for when to cheat on your principles for speed, the dark art of getting things done")</a><li><a href=/blog/2018-02-22-nylander-awkblast.html>[but how do i...](&lt;> "permalink for but how do i...")</a></ul><li><a href=/covid19>Nbis covid19 information</a><li><a href=/news>News</a><li><a href=/support>Nbis support services</a><ul><li><a href=/support/annotation>The nbis assembly and annotation service</a><li><a href=/support/biif>Bioimage informatics</a><li><a href=/support/consultation>Consultation - feedback on your project</a><li><a href=/support/dropin>Bioinformatics drop-in sessions</a><li><a href=/support/general-processing-agreements>General processing agreements for bioinformatics support and data storage</a><li><a href=/support/human-data>Working with human data</a><li><a href=/support/longtermsupport>Bioinformatics long-term support</a><li><a href=/support/ltsprojects>Long-term support projects</a><li><a href=/support/partnerprojects>Partner projects</a><li><a href=/support/support>Nbis support services</a><ul><li><a href=/support/support/support.html>Support - help with analyses</a></ul></ul><li><a href=/training>Nbis training</a><ul><li><a href=/training/coc>Nbis training code of conduct</a><li><a href=/training/events>Nbis courses and workshops</a><li><a href=/training/vision>Vision and mission for nbis training</a></ul></ul></nav></section></nav></section></section><div class=mobile-menu><div class=button_container id=toggle><span class=top></span>
<span class=middle></span>
<span class=bottom></span></div></div><section id=start><section id=body-wrapper class=section><section class="container grid-lg"><h1>Tools</h1><p>The current tools under development and maintenance by NBIS staff are listed below. For more detailed information on the tools, follow the links. See also the <a href="/infrastructure/tools/<http://opensource.scilifelab.se>">SciLifeLab Open Source website</a> for tools developed at other SciLifeLab platforms.<p>AGAT<br>URL: <a href=https://github.com/NBISweden/AGAT>https://github.com/NBISweden/AGAT</a><p>Another Gff Analysis Toolkit. Suite of tools to handle gene annotations in any GTF/GFF format.<p>Software type: Command-line tool<br>Platform: Linux, Mac, Windows<br>Development status: Maintained<br>Released: Yes<br>Affiliation: nbis.se<p>ChIP-seq QC pipeline<br>URL: <a href=https://github.com/ewels/clusterflow>https://github.com/ewels/clusterflow</a><p>Pipeline for quality control of ChIP-seq data, implemented as a plug-in for the workflow manager ClusterFlow. See pipelines/chipseq_qc.config in the ClusterFlow repository on GitHub.<p>Software type: Workflow<br>Platform: Linux<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:jakub.westholm@scilifelab.se>">jakub.westholm@scilifelab.se</a><br>Affiliation: nbis.se<p>EMBLmyGFF3<br>URL: <a href=https://github.com/NBISweden/EMBLmyGFF3>https://github.com/NBISweden/EMBLmyGFF3</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1186/s13104-018-3686-x>">Norling, Jareborg & Dainat, 2018</a><p>Software to convert GFF3 and fasta to legal EMBL format suitable for the submission to the European Nucleotide Archive (ENA) database.<p>Software type: Command-line tool<br>Platform: Linux, Mac<br>Development status: Maintained<br>Released: Yes<br>Affiliation: nbis.se<p>GUESSmyLT<br>URL: <a href=https://github.com/NBISweden/GUESSmyLT>https://github.com/NBISweden/GUESSmyLT</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.21105/joss.01344>">Berner Wik, Olin, Vigetun Haughey, Klasson & Dainat, 2019</a><p>Software to guess the RNA-Seq library type of paired and single end read files using mapping and gene annotation.<p>Software type: Command-line tool<br>Platform: Linux, Mac, Windows<br>Development status: Maintained<br>Released: Yes<br>Affiliation: nbis.se<p>IgDiscover<br>URL: <a href=http://igdiscover.se/>http://igdiscover.se/</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1038/ncomms13642>">Corcoran et al., 2016</a><p>IgDiscover analyzes antibody repertoires and discovers new V genes from high-throughput sequencing reads. Heavy chains, kappa and lambda light chains are supported (to discover VH, VK and VL genes).<p>Software type: Command-line tool<br>Platform: Linux, Mac<br>Development status: Active<br>Released: Yes<br>Contact: <a href=http://igdiscover.se/>http://igdiscover.se/</a><br>PI: Gunilla Karlsson Hedestam<br>Affiliation: ki.se<p>Metabolic Atlas<br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1073/pnas.2102344118>">Wang et al., 2021</a> , <a href="/infrastructure/tools/<http://dx.doi.org/10.1126/scisignal.aaz1482>">Robinson et al., 2020</a><p>Metabolic Atlas integrates open source genome-scale metabolic models (GEMs) of human and yeast for easy browsing and analysis.<p>Software type: Database portal, Web application, Web API, Web service<br>Development status: Active<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:contact@metabolicatlas.org>">contact@metabolicatlas.org</a><br>PI: Jens Nielsen<br>Affiliation: chalmers.se<p>MrBayes<br>URL: <a href=http://nbisweden.github.io/MrBayes/>http://nbisweden.github.io/MrBayes/</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1093/bioinformatics/17.8.754>">Huelsenbeck & Ronquist, 2001</a><p>MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.<p>Software type: Command-line tool<br>Platform: Linux, Windows, Mac<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href=http://nbisweden.github.io/MrBayes/>http://nbisweden.github.io/MrBayes/</a><br>PI: Fredrik Ronquist<br>Affiliation: nrm.se<p>PRECOG<br>URL: <a href=http://precog.lundberg.gu.se/>http://precog.lundberg.gu.se/</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1186/s12859-016-1134-2>">Fernandez-Ricaud et al., 2016</a><p>PRECOG is a tool developed to automate visualization, analysis and exploration of complex and highly resolved microbial growth data, as well as standardized extraction of the fitness components it contains. We hope that this tool will promote simplicity, transparency and standardization in microbial growth phenomics, and provide a portal with the standalone software and online tools so that other microbiology labs can easily upload, assess and visualize their growth data.<p>Software type: Desktop application, Command-line tool, Web application<br>Platform: Windows<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:luciano.fernandez@marine.gu.se>">luciano.fernandez@marine.gu.se</a><br>PI: Anders Blomberg<br>Affiliation: gu.se<p>Proteios<br>URL: <a href=http://www.proteios.org>http://www.proteios.org</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1093/bioinformatics/bti291>">Garden et al., 2005</a><p>A multi-user platform for analysis and management of proteomics data.<p>Software type: Web application<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href=http://ms-utils.org>http://ms-utils.org</a><br>PI: Fredrik Levander<br>Affiliation: immun.lth.se<p>Sarek<br>URL: <a href=https://nf-co.re/sarek>https://nf-co.re/sarek</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.12688/f1000research.16665.2>">Garcia M al. 2020</a><p>Sarek is a workflow designed to detect variants on whole genome or targeted sequencing data. Initially designed for Human, and Mouse, it can work on any species with a reference genome. Sarek can also handle tumour / normal pairs and could include additional relapses. Previously known as Cancer Analysis Workflow (CAW).<p>Software type: Command-line tool, Workflow<br>Platform: Linux<br>Development status: Active<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:maxime.garcia@scilifelab.se>">maxime.garcia@scilifelab.se</a><br>Affiliation: scilifelab.se<p>SweFreq<br>URL: <a href=https://swefreq.nbis.se>https://swefreq.nbis.se</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1038/ejhg.2017.130>">Ameur et al., 2017</a><p>Server hosting whole-genome variant frequencies for 1000 Swedish individuals generated within the SweGen project.<p>Software type: Web application<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:swefreq@scilifelab.se>">swefreq@scilifelab.se</a><br>PI: Ulf Gyllensten<br>Affiliation: scilifelab.se<p>The Human Protein Atlas<br>URL: <a href=http://www.proteinatlas.org/>http://www.proteinatlas.org/</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1038/nbt1210-1248>">Uhlen et al., 2010</a> , <a href="/infrastructure/tools/<http://dx.doi.org/10.1126/science.1260419>">Uhlen et al., 2015</a><p>Database with millions of high-resolution images. Expression and localization of proteins in a large variety of normal human tissues, cancer cells and cell lines with the aid of immunohistochemistry (IHC) images and immunofluorescence (IF) confocal microscopy images.<p>Software type: Database portal<br>Development status: Active<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:contact@proteinatlas.org>">contact@proteinatlas.org</a><br>PI: Mathias Uhlén<br>Affiliation: kth.se<br>Used in: <a href="/infrastructure/tools/<https://dx.doi.org/10.1096/fj.14-250555>">Kampf et al., 2014</a> , <a href="/infrastructure/tools/<https://dx.doi.org/10.1186/s12864-015-1686-y>">Lindskog et al., 2015</a> , <a href="/infrastructure/tools/<https://dx.doi.org/10.15252/msb.20167144>">Edfors et al., 2016</a><p>Topcons<br>URL: <a href=https://topcons.net/>https://topcons.net/</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1093/nar/gkv485>">Tsirigos et al., 2015</a><p>Consensus prediction of membrane protein topology and signal peptides<p>Software type: Web application<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href=http://bioinfo.se/>http://bioinfo.se/</a><br>PI: Arne Elofsson<br>Affiliation: dbb.su.se<p>wgs-structvar<br>URL: <a href=https://github.com/NBISweden/wgs-structvar>https://github.com/NBISweden/wgs-structvar</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1038/ejhg.2017.130>">Ameur et al., 2017</a><p>A workflow for standardized calling and filtering of structural variants, designed for whole genome sequencing projects.<p>Software type: Command-line tool<br>Platform: Linux<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href=https://nbis.se>https://nbis.se</a><br>PI:<br>Affiliation: nbis.se<p>WhatsHap<br>URL: <a href=http://whatshap.readthedocs.io>http://whatshap.readthedocs.io</a><br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1089/cmb.2014.0157>">Patterson et al., 2015</a> , <a href="/infrastructure/tools/<http://dx.doi.org/10.1101/037101>">Garg et al., 2016</a><p>WhatsHap is a software for phasing genomic variants using DNA sequencing reads, also called haplotype assembly. It is especially suitable for long reads, but works also well with short reads.<p>Software type: Command-line tool<br>Platform: Linux, Mac<br>Development status: Active<br>Released: Yes<br>Contact: <a href=https://lists.cwi.nl/mailman/listinfo/whatshap>https://lists.cwi.nl/mailman/listinfo/whatshap</a><br>PI: Tobias Marschall<br>Affiliation: mpi-inf.mpg.de<p>Xferticket<br>URL: <a href=https://xferticket.nbis.se>https://xferticket.nbis.se</a><p>Service for facilitating transfer of big datasets to and from nbis experts.<p>Software type: Web application<br>Development status: Maintained<br>Released: Yes<br>Contact: <a href="/infrastructure/tools/<mailto:cto@nbis.se>">cto@nbis.se</a><br>Affiliation: nbis.se<p>contigtax UNDER DEVELOPMENT<br>URL: <a href=https://github.com/NBISweden/contigtax>https://github.com/NBISweden/contigtax</a><p>contigtax assigns taxonomy to contigs in metagenomic assemblies using sequence alignments against protein databases (e.g. UniRef) and calculation of lowest common ancestor in the taxonomic hierarchy.<p>Software type: Command-line tool<br>Platform: Linux, Mac<br>Development status: Active<br>Released: No<br>Contact: <a href="/infrastructure/tools/<mailto:john.sundh@nbis.se>">john.sundh@nbis.se</a><br>Affiliation: nbis.se<p>HaloPlex variant calling pipeline UNDER DEVELOPMENT<br>Publications: <a href="/infrastructure/tools/<http://dx.doi.org/10.1002/gcc.22363>">Tuominen et al., 2016</a> , <a href="/infrastructure/tools/<http://dx.doi.org/10.1016/j.ijcard.2016.11.066>">Dalin et al., 2017</a><p>A pipeline for calling variants from targeted DNA sequencing data obtained using the HaloPlex method.<p>Software type: Workflow<br>Platform: Linux<br>Development status: Active<br>Released: No<br>Contact: <a href="/infrastructure/tools/<mailto:par.engstrom@scilifelab.se>">par.engstrom@scilifelab.se</a><br>Affiliation: nbis.se<p>LocalEGA UNDER DEVELOPMENT<br>URL: <a href=https://github.com/NBISweden/LocalEGA>https://github.com/NBISweden/LocalEGA</a><p>The European Genome-phenome Archive (EGA) is a service for permanent archiving and sharing of all types of personally identifiable genetic and phenotypic data resulting from biomedical research projects. Local EGA handles data that are supposed to stay in Sweden.<p>Software type: Command-line tool, Workflow<br>Platform: Linux, Mac<br>Development status: Active<br>Released: No<br>Contact: <a href="/infrastructure/tools/<mailto:ega@nbis.se>">ega@nbis.se</a><br>Affiliation: nbis.se<p>Single-cell RNA-seq analysis workflow UNDER DEVELOPMENT<br>URL: <a href=https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop>https://bitbucket.org/scilifelab-lts/lts-workflows-sm-scrnaseq/branch/develop</a><p>A workflow for quality control and basic analysis of single-cell RNA-seq data.<p>Software type: Workflow<br>Platform: Linux<br>Development status: Active<br>Released: No<br>Contact: <a href="/infrastructure/tools/<mailto:asa.bjorklund@scilifelab.se>">asa.bjorklund@scilifelab.se</a><br>Affiliation: nbis.se</section></section></section></div><section id=footer class="section bg-gray"><section class="container grid-lg"><p><a href=http://getgrav.org>Grav</a> was <i class="fa fa-code"></i> with <i class="fa fa-heart-o pulse"></i> by <a href=https://trilby.media>Trilby Media</a>.</section></section><div class=mobile-container><div class=overlay id=overlay><div class=mobile-logo><a href=/ class="navbar-brand mr-10"><svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" viewBox="0 0 504 140" clip-rule="evenodd"><path d="M235.83 71.56h-7.98c-1.2.0-2.2 1-2.2 2.2V89.1l-.15.13c-4.7 3.96-10.64 6.14-16.72 6.14-14.36.0-26.04-11.68-26.04-26.04s11.68-26.04 26.04-26.04c5.58.0 10.92 1.76 15.44 5.1.87.66 2.1.57 2.86-.2l5.7-5.7c.44-.44.67-1.05.63-1.68-.02-.62-.32-1.2-.82-1.6-6.76-5.35-15.2-8.3-23.8-8.3-21.18.0-38.42 17.23-38.42 38.4.0 21.2 17.24 38.42 38.42 38.42 10.93.0 21.4-4.7 28.7-12.9.35-.4.55-.93.55-1.47v-19.6c0-1.22-.98-2.2-2.2-2.2M502.8 34.44c-.4-.6-1.1-.98-1.84-.98h-8.7c-.87.0-1.66.52-2 1.32l-24.5 56.84-24.9-56.85c-.36-.8-1.15-1.3-2.02-1.3h-8.72c-.74.0-1.44.36-1.84.98s-.48 1.4-.17 2.1l30.2 68.85c.34.8 1.13 1.32 2 1.32h11c.88.0 1.67-.53 2.02-1.33l29.66-68.87c.3-.68.22-1.47-.2-2.1"/><path d="M388.68 34.77c-.35-.8-1.14-1.32-2-1.32h-11c-.88.0-1.67.53-2.02 1.33L344 103.64c-.3.68-.22 1.47.18 2.08.4.62 1.1 1 1.84 1h8.7c.86.0 1.66-.53 2-1.33l24.5-56.86 24.9 56.86c.36.8 1.15 1.32 2.02 1.32h8.72c.74.0 1.44-.38 1.84-1s.47-1.4.17-2.1l-30.2-68.85zM309.2 81.52l.47-.22c8.68-4.2 14.28-13.1 14.28-22.67.0-13.88-11.3-25.18-25.17-25.18H266.9c-1.2.0-2.2 1-2.2 2.2v68.86c0 1.23 1 2.22 2.2 2.22h8c1.2.0 2.2-1 2.2-2.2V45.8h21.68c7.05.0 12.8 5.75 12.8 12.8.0 5.9-4 11-9.73 12.42-1.04.26-2.07.4-3.07.4h-7.98c-.83.0-1.6.46-1.96 1.2-.38.73-.3 1.62.2 2.3l22.6 30.87c.42.58 1.08.92 1.78.92h9.9c.84.0 1.6-.47 1.97-1.2.37-.75.3-1.64-.2-2.3l-15.9-21.7zM107.2 80.97c-7.26-4.8-11.4-8.85-15.02-16.1-2.47 4.97-8.24 12.37-17.96 18.2-4.86 15.1-27.96 44-35.43 39.9-2.22-1.2-2.64-2.8-2.15-4.45.54-4.13 9.08-13.62 9.08-13.62s.18 2 2.92 6.18c-3.6-11.2 5.96-25.03 8.5-29.73 3.98-1.27 4.27-6.4 4.27-6.4.26-7.9-3.28-13.63-6.7-17.05 2.46 3 3.25 7.54 3.37 11.7v.02c0 .47.0.93.0 1.4-.12 3.43-1.16 8.18-3.38 8.18v.03c-2.28-.1-5.1.4-7.63 1.18l-5.6 1.34s2.98-.13 4.6 1.25c-1.8 2.9-5.78 6.53-10.22 8.58-6.45 3-8.3-2.96-5.03-6.84.8-.94 1.62-1.74 2.38-2.4-.5-.5-.8-1.2-.88-2.06v-.02c-.46-1.97-.2-4.54 2.6-8.62.54-.86 1.2-1.75 2-2.65.02-.04.04-.07.07-.1.03-.04.07-.08.1-.12.02-.02.04-.04.06-.06.2-.23.42-.45.64-.67 3.34-3.4 8.6-6.96 16.9-10.15C64.4 43.68 67.94 41 67.94 41c1.07-1.1 2.94-2.45 3.63-2.8-5.05-8.77-6.07-21.15-4.75-24.5-.1.2-.2.38-.3.57.5-1.14.83-1.5 1.34-2.1 1.38-1.64 6.06-2.5 7.74.96.9 1.84 1.06 4.23 1.03 6.02-3.7-.2-7.06 4.04-7.06 4.04s3.07-1.46 6.88-1.5c0 0 1 .9 2.28 2.56-1.7 3.2-4.52 10.02-2.5 17.16.35 1.4.86 2.62 1.5 3.65.02.05.04.1.07.14.05.07.1.13.14.2 3.37 5.06 9.54 5.66 9.54 5.66-2.9-1.45-5.27-3.76-6.8-6.56-.82-1.5-1.3-2.77-1.6-3.77-1.64-6.3.77-10 2.14-12.47 3.17-4.9 8.95-7.9 15.15-7.18 8.72 1 14.97 8.86 13.98 17.57-.6 5.32-3.78 9.72-8.15 12.12 1.05 2.84-.07 6.28-.07 6.28 2.64 3.32 2.76 5.23 2.67 7-3.36-.55-6.62 1.7-6.62 1.7s6.48-1.53 10.24 1.82c2.44 2.64 4.08 5 5.05 6.77 1.4 2.5 7.86 2.68 7.12 7.2-.74 4.5-5.68 4.53-13.4-.57M69.56.0C31.15.0.0 31.15.0 69.57c0 38.42 31.15 69.57 69.57 69.57 38.42.0 69.57-31.15 69.57-69.57C139.14 31.15 108 0 69.57.0M73.8 51.7c.8-.82.8-2.14.0-2.95-.82-.82-2.14-.82-2.95.0-.82.8-.82 2.13.0 2.94.8.8 2.13.8 2.95.0M66.45 53.15c-.82.8-.82 2.13.0 2.95.8.8 2.13.8 2.94.0.8-.82.8-2.14.0-2.95-.82-.8-2.14-.8-2.95.0"/><path d="M79.23 54.23c-1.27-1.27-3.34-1.27-4.6.0l-2.72 2.7c-1.27 1.3-1.27 3.35.0 4.63l3 2.97c1.26 1.28 3.32 1.28 4.6.0l2.7-2.7c1.28-1.28 1.28-3.35.0-4.62l-2.97-2.97zM95.76 41.44c-2.15-2.57 1.87-7.25 4.4-4.46 4.64 5.15-2.25 7.04-4.4 4.46m9.24 2.7c3.45-6.56-1.42-10.4-4.77-13.53-5.36-5.03-10.7-7.2-16.8-.23-6.1 6.98-2.24 15.07 3.35 19.06 5.58 4 14.78 1.25 18.22-5.3"/></svg></a></div><nav class=overlay-menu><ul class=tree><li><a href=/>Home</a><li><a href=/infrastructure class=active>Nbis infrastructure</a><ul><li><a href=/infrastructure/aida-data-hub>Aida data hub</a><li><a href=/infrastructure/compute-and-storage>Nbis compute and storage</a><li><a href=/infrastructure/data-management>Data management</a><ul><li><a href=/infrastructure/data-management/data-management.html>Data management</a></ul><li><a href=/infrastructure/general-information>General information</a><li><a href=/infrastructure/mosler>Mosler</a><li><a href=/infrastructure/sensitive-data-archive>Swedish sensitive data archive</a><li><a href=/infrastructure/tools class=active>Tools</a></ul><li><a href=/events>Events</a><ul><li><a href=/events/past-events>Previous nbis events</a><li><a href=/events/symposium2018>Nbis symposium and user meeting 2018</a></ul><li><a href=/search>Search</a><li><a href=/about>About nbis</a><ul><li><a href=/about/contact>Contact information</a><li><a href=/about/doc>Nbis documents</a><li><a href=/about/organization>Nbis organization</a><li><a href=/about/privacy>Privacy policy</a><li><a href=/about/staff>Nbis staff</a><li><a href=/about/status>Current status</a><li><a href=/about/vacancies>Vacancies</a></ul><li><a href=/blog>Blog</a><ul><li><a href=/blog/2018-02-22-cheating-for-speed.html>[when to cheat on your principles for speed, the dark art of getting things done](&lt;> "permalink for when to cheat on your principles for speed, the dark art of getting things done")</a><li><a href=/blog/2018-02-22-nylander-awkblast.html>[but how do i...](&lt;> "permalink for but how do i...")</a></ul><li><a href=/covid19>Nbis covid19 information</a><li><a href=/news>News</a><li><a href=/support>Nbis support services</a><ul><li><a href=/support/annotation>The nbis assembly and annotation service</a><li><a href=/support/biif>Bioimage informatics</a><li><a href=/support/consultation>Consultation - feedback on your project</a><li><a href=/support/dropin>Bioinformatics drop-in sessions</a><li><a href=/support/general-processing-agreements>General processing agreements for bioinformatics support and data storage</a><li><a href=/support/human-data>Working with human data</a><li><a href=/support/longtermsupport>Bioinformatics long-term support</a><li><a href=/support/ltsprojects>Long-term support projects</a><li><a href=/support/partnerprojects>Partner projects</a><li><a href=/support/support>Nbis support services</a><ul><li><a href=/support/support/support.html>Support - help with analyses</a></ul></ul><li><a href=/training>Nbis training</a><ul><li><a href=/training/coc>Nbis training code of conduct</a><li><a href=/training/events>Nbis courses and workshops</a><li><a href=/training/vision>Vision and mission for nbis training</a></ul></ul></nav></div></div><script src=/user/themes/quark/js/jquery.treemenu.js></script>
<script src=/user/themes/quark/js/site.js></script>